Time taken to run the queries on ./repositories/Django/thread4/3chamchi_project-lion-backend-django/3chamchi-project-lion-backend-django-8e03b70: 2924.708 seconds.
Analysis failed for: ./repositories/Django/thread4/AaLondon_aalondon/AaLondon-aalondon-aa4f582 Reason: Error: Command failed: npm start -- -s=./repositories/Django/thread4/AaLondon_aalondon/AaLondon-aalondon-aa4f582 -l=python -t=11 -ct=4
Compiling query plan for /home/878352/Session-Management-Analyser/Flask_Queries/Flask-login-is-used-check/flask_library_used_check.ql.
[1/1] Found in cache: /home/878352/Session-Management-Analyser/Flask_Queries/Flask-login-is-used-check/flask_library_used_check.ql.
Starting evaluation of asterius27/codeql-extra-queries-python/Flask_Queries/Flask-login-is-used-check/flask_library_used_check.ql.
[1/1 eval 2m13s] Evaluation done; writing results to /home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/repositories/Django/thread4/AaLondon_aalondon/AaLondon-aalondon-aa4f582-results/flask_library_used_check.bqrs.
Shutting down query evaluator.
Compiling query plan for /home/878352/Session-Management-Analyser/Django_Queries/Django-auth-is-used-check/django_library_used_check.ql.
[1/1] Found in cache: /home/878352/Session-Management-Analyser/Django_Queries/Django-auth-is-used-check/django_library_used_check.ql.
Starting evaluation of asterius27/codeql-extra-queries-python/Django_Queries/Django-auth-is-used-check/django_library_used_check.ql.
[1/1 eval 21.9s] Evaluation done; writing results to /home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/repositories/Django/thread4/AaLondon_aalondon/AaLondon-aalondon-aa4f582-results/django_library_used_check.bqrs.
Shutting down query evaluator.
file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/python-single-analysis.js:97
        throw new Error("None of the supported libraries/frameworks is used");
              ^

Error: None of the supported libraries/frameworks is used
    at pythonAnalysis (file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/python-single-analysis.js:97:15)
    at file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/app.js:57:5
    at ModuleJob.run (node:internal/modules/esm/module_job:217:25)
    at async ModuleLoader.import (node:internal/modules/esm/loader:316:24)
    at async loadESM (node:internal/process/esm_loader:34:7)
    at async handleMainPromise (node:internal/modules/run_main:66:12)

Node.js v20.8.0

Time taken to run the queries on ./repositories/Django/thread4/AaLondon_aalondon/AaLondon-aalondon-aa4f582: 388.782 seconds.
Analysis failed for: ./repositories/Django/thread4/Alschn_MusicRooms/Alschn-MusicRooms-099afad Reason: Error: Command failed: npm start -- -s=./repositories/Django/thread4/Alschn_MusicRooms/Alschn-MusicRooms-099afad -l=python -t=11 -ct=4
Compiling query plan for /home/878352/Session-Management-Analyser/Flask_Queries/Flask-login-is-used-check/flask_library_used_check.ql.
[1/1] Found in cache: /home/878352/Session-Management-Analyser/Flask_Queries/Flask-login-is-used-check/flask_library_used_check.ql.
Starting evaluation of asterius27/codeql-extra-queries-python/Flask_Queries/Flask-login-is-used-check/flask_library_used_check.ql.
[1/1 eval 1m47s] Evaluation done; writing results to /home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/repositories/Django/thread4/Alschn_MusicRooms/Alschn-MusicRooms-099afad-results/flask_library_used_check.bqrs.
Shutting down query evaluator.
Compiling query plan for /home/878352/Session-Management-Analyser/Django_Queries/Django-auth-is-used-check/django_library_used_check.ql.
[1/1] Found in cache: /home/878352/Session-Management-Analyser/Django_Queries/Django-auth-is-used-check/django_library_used_check.ql.
Starting evaluation of asterius27/codeql-extra-queries-python/Django_Queries/Django-auth-is-used-check/django_library_used_check.ql.
[1/1 eval 18.2s] Evaluation done; writing results to /home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/repositories/Django/thread4/Alschn_MusicRooms/Alschn-MusicRooms-099afad-results/django_library_used_check.bqrs.
Shutting down query evaluator.
file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/python-single-analysis.js:97
        throw new Error("None of the supported libraries/frameworks is used");
              ^

Error: None of the supported libraries/frameworks is used
    at pythonAnalysis (file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/python-single-analysis.js:97:15)
    at file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/app.js:57:5
    at ModuleJob.run (node:internal/modules/esm/module_job:217:25)
    at async ModuleLoader.import (node:internal/modules/esm/loader:316:24)
    at async loadESM (node:internal/process/esm_loader:34:7)
    at async handleMainPromise (node:internal/modules/run_main:66:12)

Node.js v20.8.0

Time taken to run the queries on ./repositories/Django/thread4/Alschn_MusicRooms/Alschn-MusicRooms-099afad: 318.41 seconds.
Analysis failed for: ./repositories/Django/thread4/Bearle_django_private_chat2/Bearle-django_private_chat2-1845878 Reason: Error: Command failed: npm start -- -s=./repositories/Django/thread4/Bearle_django_private_chat2/Bearle-django_private_chat2-1845878 -l=python -t=11 -ct=4
Compiling query plan for /home/878352/Session-Management-Analyser/Flask_Queries/Flask-login-is-used-check/flask_library_used_check.ql.
[1/1] Found in cache: /home/878352/Session-Management-Analyser/Flask_Queries/Flask-login-is-used-check/flask_library_used_check.ql.
Starting evaluation of asterius27/codeql-extra-queries-python/Flask_Queries/Flask-login-is-used-check/flask_library_used_check.ql.
[1/1 eval 1m48s] Evaluation done; writing results to /home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/repositories/Django/thread4/Bearle_django_private_chat2/Bearle-django_private_chat2-1845878-results/flask_library_used_check.bqrs.
Shutting down query evaluator.
Compiling query plan for /home/878352/Session-Management-Analyser/Django_Queries/Django-auth-is-used-check/django_library_used_check.ql.
[1/1] Found in cache: /home/878352/Session-Management-Analyser/Django_Queries/Django-auth-is-used-check/django_library_used_check.ql.
Starting evaluation of asterius27/codeql-extra-queries-python/Django_Queries/Django-auth-is-used-check/django_library_used_check.ql.
[1/1 eval 12.9s] Evaluation done; writing results to /home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/repositories/Django/thread4/Bearle_django_private_chat2/Bearle-django_private_chat2-1845878-results/django_library_used_check.bqrs.
Shutting down query evaluator.
file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/python-single-analysis.js:97
        throw new Error("None of the supported libraries/frameworks is used");
              ^

Error: None of the supported libraries/frameworks is used
    at pythonAnalysis (file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/python-single-analysis.js:97:15)
    at file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/app.js:57:5
    at ModuleJob.run (node:internal/modules/esm/module_job:217:25)
    at async ModuleLoader.import (node:internal/modules/esm/loader:316:24)
    at async loadESM (node:internal/process/esm_loader:34:7)
    at async handleMainPromise (node:internal/modules/run_main:66:12)

Node.js v20.8.0

Time taken to run the queries on ./repositories/Django/thread4/Bearle_django_private_chat2/Bearle-django_private_chat2-1845878: 262.448 seconds.
Time taken to run the queries on ./repositories/Django/thread4/Blueshoe_pycloak/Blueshoe-pycloak-08ee3be: 1891.046 seconds.
Analysis failed for: ./repositories/Django/thread4/CN-UPB_Pishahang/CN-UPB-Pishahang-28a6852 Reason: Error: Command failed: npm start -- -s=./repositories/Django/thread4/CN-UPB_Pishahang/CN-UPB-Pishahang-28a6852 -l=python -t=11 -ct=4
Compiling query plan for /home/878352/Session-Management-Analyser/Flask_Queries/Flask-login-is-used-check/flask_library_used_check.ql.
[1/1] Found in cache: /home/878352/Session-Management-Analyser/Flask_Queries/Flask-login-is-used-check/flask_library_used_check.ql.
Starting evaluation of asterius27/codeql-extra-queries-python/Flask_Queries/Flask-login-is-used-check/flask_library_used_check.ql.
[1/1 eval 2m30s] Evaluation done; writing results to /home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/repositories/Django/thread4/CN-UPB_Pishahang/CN-UPB-Pishahang-28a6852-results/flask_library_used_check.bqrs.
Shutting down query evaluator.
Compiling query plan for /home/878352/Session-Management-Analyser/Django_Queries/Django-auth-is-used-check/django_library_used_check.ql.
[1/1] Found in cache: /home/878352/Session-Management-Analyser/Django_Queries/Django-auth-is-used-check/django_library_used_check.ql.
Starting evaluation of asterius27/codeql-extra-queries-python/Django_Queries/Django-auth-is-used-check/django_library_used_check.ql.
[1/1 eval 18.3s] Evaluation done; writing results to /home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/repositories/Django/thread4/CN-UPB_Pishahang/CN-UPB-Pishahang-28a6852-results/django_library_used_check.bqrs.
Shutting down query evaluator.
file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/python-single-analysis.js:97
        throw new Error("None of the supported libraries/frameworks is used");
              ^

Error: None of the supported libraries/frameworks is used
    at pythonAnalysis (file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/python-single-analysis.js:97:15)
    at file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/app.js:57:5
    at ModuleJob.run (node:internal/modules/esm/module_job:217:25)
    at async ModuleLoader.import (node:internal/modules/esm/loader:316:24)
    at async loadESM (node:internal/process/esm_loader:34:7)
    at async handleMainPromise (node:internal/modules/run_main:66:12)

Node.js v20.8.0

Time taken to run the queries on ./repositories/Django/thread4/CN-UPB_Pishahang/CN-UPB-Pishahang-28a6852: 327.777 seconds.
Time taken to run the queries on ./repositories/Django/thread4/CancerGary_sdorica-inspector/CancerGary-sdorica-inspector-1c5f5eb: 2361.237 seconds.
Analysis failed for: ./repositories/Django/thread4/Ceci-Aguilera_truck_signs_api/Ceci-Aguilera-truck_signs_api-8d29bbf Reason: Error: Command failed: npm start -- -s=./repositories/Django/thread4/Ceci-Aguilera_truck_signs_api/Ceci-Aguilera-truck_signs_api-8d29bbf -l=python -t=11 -ct=4
Compiling query plan for /home/878352/Session-Management-Analyser/Flask_Queries/Flask-login-is-used-check/flask_library_used_check.ql.
[1/1] Found in cache: /home/878352/Session-Management-Analyser/Flask_Queries/Flask-login-is-used-check/flask_library_used_check.ql.
Starting evaluation of asterius27/codeql-extra-queries-python/Flask_Queries/Flask-login-is-used-check/flask_library_used_check.ql.
[1/1 eval 1m9s] Evaluation done; writing results to /home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/repositories/Django/thread4/Ceci-Aguilera_truck_signs_api/Ceci-Aguilera-truck_signs_api-8d29bbf-results/flask_library_used_check.bqrs.
Shutting down query evaluator.
Compiling query plan for /home/878352/Session-Management-Analyser/Django_Queries/Django-auth-is-used-check/django_library_used_check.ql.
[1/1] Found in cache: /home/878352/Session-Management-Analyser/Django_Queries/Django-auth-is-used-check/django_library_used_check.ql.
Starting evaluation of asterius27/codeql-extra-queries-python/Django_Queries/Django-auth-is-used-check/django_library_used_check.ql.
[1/1 eval 9.7s] Evaluation done; writing results to /home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/repositories/Django/thread4/Ceci-Aguilera_truck_signs_api/Ceci-Aguilera-truck_signs_api-8d29bbf-results/django_library_used_check.bqrs.
Shutting down query evaluator.
file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/python-single-analysis.js:97
        throw new Error("None of the supported libraries/frameworks is used");
              ^

Error: None of the supported libraries/frameworks is used
    at pythonAnalysis (file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/python-single-analysis.js:97:15)
    at file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/app.js:57:5
    at ModuleJob.run (node:internal/modules/esm/module_job:217:25)
    at async ModuleLoader.import (node:internal/modules/esm/loader:316:24)
    at async loadESM (node:internal/process/esm_loader:34:7)
    at async handleMainPromise (node:internal/modules/run_main:66:12)

Node.js v20.8.0

Time taken to run the queries on ./repositories/Django/thread4/Ceci-Aguilera_truck_signs_api/Ceci-Aguilera-truck_signs_api-8d29bbf: 202.906 seconds.
Analysis failed for: ./repositories/Django/thread4/City-of-Helsinki_open-city-profile/City-of-Helsinki-open-city-profile-689e4d7 Reason: Error: Command failed: npm start -- -s=./repositories/Django/thread4/City-of-Helsinki_open-city-profile/City-of-Helsinki-open-city-profile-689e4d7 -l=python -t=11 -ct=4
Compiling query plan for /home/878352/Session-Management-Analyser/Flask_Queries/Flask-login-is-used-check/flask_library_used_check.ql.
[1/1] Found in cache: /home/878352/Session-Management-Analyser/Flask_Queries/Flask-login-is-used-check/flask_library_used_check.ql.
Starting evaluation of asterius27/codeql-extra-queries-python/Flask_Queries/Flask-login-is-used-check/flask_library_used_check.ql.
[1/1 eval 1m27s] Evaluation done; writing results to /home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/repositories/Django/thread4/City-of-Helsinki_open-city-profile/City-of-Helsinki-open-city-profile-689e4d7-results/flask_library_used_check.bqrs.
Shutting down query evaluator.
Compiling query plan for /home/878352/Session-Management-Analyser/Django_Queries/Django-auth-is-used-check/django_library_used_check.ql.
[1/1] Found in cache: /home/878352/Session-Management-Analyser/Django_Queries/Django-auth-is-used-check/django_library_used_check.ql.
Starting evaluation of asterius27/codeql-extra-queries-python/Django_Queries/Django-auth-is-used-check/django_library_used_check.ql.
[1/1 eval 15.4s] Evaluation done; writing results to /home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/repositories/Django/thread4/City-of-Helsinki_open-city-profile/City-of-Helsinki-open-city-profile-689e4d7-results/django_library_used_check.bqrs.
Shutting down query evaluator.
file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/python-single-analysis.js:97
        throw new Error("None of the supported libraries/frameworks is used");
              ^

Error: None of the supported libraries/frameworks is used
    at pythonAnalysis (file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/python-single-analysis.js:97:15)
    at file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/app.js:57:5
    at ModuleJob.run (node:internal/modules/esm/module_job:217:25)
    at async ModuleLoader.import (node:internal/modules/esm/loader:316:24)
    at async loadESM (node:internal/process/esm_loader:34:7)
    at async handleMainPromise (node:internal/modules/run_main:66:12)

Node.js v20.8.0

Time taken to run the queries on ./repositories/Django/thread4/City-of-Helsinki_open-city-profile/City-of-Helsinki-open-city-profile-689e4d7: 241.796 seconds.
Time taken to run the queries on ./repositories/Django/thread4/CodingMantras_Django-Advanced-Boilerplate/CodingMantras-Django-Advanced-Boilerplate-497854b: 1714.73 seconds.
Time taken to run the queries on ./repositories/Django/thread4/DAVIDhaker_django-sso/DAVIDhaker-django-sso-a6fe1f1: 1872.008 seconds.
Analysis failed for: ./repositories/Django/thread4/Davidcode-png_Shosho-Media/Davidcode-png-Shosho-Media-036473c Reason: Error: Command failed: npm start -- -s=./repositories/Django/thread4/Davidcode-png_Shosho-Media/Davidcode-png-Shosho-Media-036473c -l=python -t=11 -ct=4
Compiling query plan for /home/878352/Session-Management-Analyser/Flask_Queries/Flask-login-is-used-check/flask_library_used_check.ql.
[1/1] Found in cache: /home/878352/Session-Management-Analyser/Flask_Queries/Flask-login-is-used-check/flask_library_used_check.ql.
Starting evaluation of asterius27/codeql-extra-queries-python/Flask_Queries/Flask-login-is-used-check/flask_library_used_check.ql.
[1/1 eval 1m14s] Evaluation done; writing results to /home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/repositories/Django/thread4/Davidcode-png_Shosho-Media/Davidcode-png-Shosho-Media-036473c-results/flask_library_used_check.bqrs.
Shutting down query evaluator.
Compiling query plan for /home/878352/Session-Management-Analyser/Django_Queries/Django-auth-is-used-check/django_library_used_check.ql.
[1/1] Found in cache: /home/878352/Session-Management-Analyser/Django_Queries/Django-auth-is-used-check/django_library_used_check.ql.
Starting evaluation of asterius27/codeql-extra-queries-python/Django_Queries/Django-auth-is-used-check/django_library_used_check.ql.
[1/1 eval 14.4s] Evaluation done; writing results to /home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/repositories/Django/thread4/Davidcode-png_Shosho-Media/Davidcode-png-Shosho-Media-036473c-results/django_library_used_check.bqrs.
Shutting down query evaluator.
file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/python-single-analysis.js:97
        throw new Error("None of the supported libraries/frameworks is used");
              ^

Error: None of the supported libraries/frameworks is used
    at pythonAnalysis (file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/python-single-analysis.js:97:15)
    at file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/app.js:57:5
    at ModuleJob.run (node:internal/modules/esm/module_job:217:25)
    at async ModuleLoader.import (node:internal/modules/esm/loader:316:24)
    at async loadESM (node:internal/process/esm_loader:34:7)
    at async handleMainPromise (node:internal/modules/run_main:66:12)

Node.js v20.8.0

Time taken to run the queries on ./repositories/Django/thread4/Davidcode-png_Shosho-Media/Davidcode-png-Shosho-Media-036473c: 219.265 seconds.
Analysis failed for: ./repositories/Django/thread4/Deepomatic_dmake/Deepomatic-dmake-ce27417 Reason: Error: Command failed: npm start -- -s=./repositories/Django/thread4/Deepomatic_dmake/Deepomatic-dmake-ce27417 -l=python -t=11 -ct=4
Compiling query plan for /home/878352/Session-Management-Analyser/Flask_Queries/Flask-login-is-used-check/flask_library_used_check.ql.
[1/1] Found in cache: /home/878352/Session-Management-Analyser/Flask_Queries/Flask-login-is-used-check/flask_library_used_check.ql.
Starting evaluation of asterius27/codeql-extra-queries-python/Flask_Queries/Flask-login-is-used-check/flask_library_used_check.ql.
[1/1 eval 1m40s] Evaluation done; writing results to /home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/repositories/Django/thread4/Deepomatic_dmake/Deepomatic-dmake-ce27417-results/flask_library_used_check.bqrs.
Shutting down query evaluator.
Compiling query plan for /home/878352/Session-Management-Analyser/Django_Queries/Django-auth-is-used-check/django_library_used_check.ql.
[1/1] Found in cache: /home/878352/Session-Management-Analyser/Django_Queries/Django-auth-is-used-check/django_library_used_check.ql.
Starting evaluation of asterius27/codeql-extra-queries-python/Django_Queries/Django-auth-is-used-check/django_library_used_check.ql.
[1/1 eval 11.7s] Evaluation done; writing results to /home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/repositories/Django/thread4/Deepomatic_dmake/Deepomatic-dmake-ce27417-results/django_library_used_check.bqrs.
Shutting down query evaluator.
file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/python-single-analysis.js:97
        throw new Error("None of the supported libraries/frameworks is used");
              ^

Error: None of the supported libraries/frameworks is used
    at pythonAnalysis (file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/python-single-analysis.js:97:15)
    at file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/app.js:57:5
    at ModuleJob.run (node:internal/modules/esm/module_job:217:25)
    at async ModuleLoader.import (node:internal/modules/esm/loader:316:24)
    at async loadESM (node:internal/process/esm_loader:34:7)
    at async handleMainPromise (node:internal/modules/run_main:66:12)

Node.js v20.8.0

Time taken to run the queries on ./repositories/Django/thread4/Deepomatic_dmake/Deepomatic-dmake-ce27417: 246.037 seconds.
Time taken to run the queries on ./repositories/Django/thread4/Django-LiveView_starter-template/Django-LiveView-starter-template-c5a0c11: 1939.024 seconds.
Time taken to run the queries on ./repositories/Django/thread4/Eugeny_tabby-web/Eugeny-tabby-web-abcdfbc: 1989.137 seconds.
Time taken to run the queries on ./repositories/Django/thread4/Eva-Kuk_love-for-food/Eva-Kuk-love-for-food-c517250: 1737.549 seconds.
Analysis failed for: ./repositories/Django/thread4/FundersClub_fire/FundersClub-fire-a41bbdb Reason: Error: Command failed: npm start -- -s=./repositories/Django/thread4/FundersClub_fire/FundersClub-fire-a41bbdb -l=python -t=11 -ct=4
Compiling query plan for /home/878352/Session-Management-Analyser/Flask_Queries/Flask-login-is-used-check/flask_library_used_check.ql.
[1/1] Found in cache: /home/878352/Session-Management-Analyser/Flask_Queries/Flask-login-is-used-check/flask_library_used_check.ql.
Starting evaluation of asterius27/codeql-extra-queries-python/Flask_Queries/Flask-login-is-used-check/flask_library_used_check.ql.
[1/1 eval 1m5s] Evaluation done; writing results to /home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/repositories/Django/thread4/FundersClub_fire/FundersClub-fire-a41bbdb-results/flask_library_used_check.bqrs.
Shutting down query evaluator.
Compiling query plan for /home/878352/Session-Management-Analyser/Django_Queries/Django-auth-is-used-check/django_library_used_check.ql.
[1/1] Found in cache: /home/878352/Session-Management-Analyser/Django_Queries/Django-auth-is-used-check/django_library_used_check.ql.
Starting evaluation of asterius27/codeql-extra-queries-python/Django_Queries/Django-auth-is-used-check/django_library_used_check.ql.
[1/1 eval 9.8s] Evaluation done; writing results to /home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/repositories/Django/thread4/FundersClub_fire/FundersClub-fire-a41bbdb-results/django_library_used_check.bqrs.
Shutting down query evaluator.
file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/python-single-analysis.js:97
        throw new Error("None of the supported libraries/frameworks is used");
              ^

Error: None of the supported libraries/frameworks is used
    at pythonAnalysis (file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/python-single-analysis.js:97:15)
    at file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/app.js:57:5
    at ModuleJob.run (node:internal/modules/esm/module_job:217:25)
    at async ModuleLoader.import (node:internal/modules/esm/loader:316:24)
    at async loadESM (node:internal/process/esm_loader:34:7)
    at async handleMainPromise (node:internal/modules/run_main:66:12)

Node.js v20.8.0

Time taken to run the queries on ./repositories/Django/thread4/FundersClub_fire/FundersClub-fire-a41bbdb: 178.128 seconds.
Time taken to run the queries on ./repositories/Django/thread4/Gencaster_gencaster/Gencaster-gencaster-ef7a474: 1970.619 seconds.
Analysis failed for: ./repositories/Django/thread4/GitHub-WeiChiang_main/GitHub-WeiChiang-main-a450d65 Reason: Error: Command failed: npm start -- -s=./repositories/Django/thread4/GitHub-WeiChiang_main/GitHub-WeiChiang-main-a450d65 -l=python -t=11 -ct=4
Compiling query plan for /home/878352/Session-Management-Analyser/Flask_Queries/Flask-login-is-used-check/flask_library_used_check.ql.
[1/1] Found in cache: /home/878352/Session-Management-Analyser/Flask_Queries/Flask-login-is-used-check/flask_library_used_check.ql.
Starting evaluation of asterius27/codeql-extra-queries-python/Flask_Queries/Flask-login-is-used-check/flask_library_used_check.ql.
[1/1 eval 1m51s] Evaluation done; writing results to /home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/repositories/Django/thread4/GitHub-WeiChiang_main/GitHub-WeiChiang-main-a450d65-results/flask_library_used_check.bqrs.
Shutting down query evaluator.
Compiling query plan for /home/878352/Session-Management-Analyser/Django_Queries/Django-auth-is-used-check/django_library_used_check.ql.
[1/1] Found in cache: /home/878352/Session-Management-Analyser/Django_Queries/Django-auth-is-used-check/django_library_used_check.ql.
Starting evaluation of asterius27/codeql-extra-queries-python/Django_Queries/Django-auth-is-used-check/django_library_used_check.ql.
[1/1 eval 9.6s] Evaluation done; writing results to /home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/repositories/Django/thread4/GitHub-WeiChiang_main/GitHub-WeiChiang-main-a450d65-results/django_library_used_check.bqrs.
Shutting down query evaluator.
file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/python-single-analysis.js:97
        throw new Error("None of the supported libraries/frameworks is used");
              ^

Error: None of the supported libraries/frameworks is used
    at pythonAnalysis (file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/python-single-analysis.js:97:15)
    at file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/app.js:57:5
    at ModuleJob.run (node:internal/modules/esm/module_job:217:25)
    at async ModuleLoader.import (node:internal/modules/esm/loader:316:24)
    at async loadESM (node:internal/process/esm_loader:34:7)
    at async handleMainPromise (node:internal/modules/run_main:66:12)

Node.js v20.8.0

Time taken to run the queries on ./repositories/Django/thread4/GitHub-WeiChiang_main/GitHub-WeiChiang-main-a450d65: 271.176 seconds.
Time taken to run the queries on ./repositories/Django/thread4/HeliumEdu_platform/HeliumEdu-platform-f5dae35: 1849.784 seconds.
Analysis failed for: ./repositories/Django/thread4/HumanBrainProject_hbp_neuromorphic_platform/HumanBrainProject-ebrains-neuromorphic-job-queue-api-3c0e1f1 Reason: Error: Command failed: npm start -- -s=./repositories/Django/thread4/HumanBrainProject_hbp_neuromorphic_platform/HumanBrainProject-ebrains-neuromorphic-job-queue-api-3c0e1f1 -l=python -t=11 -ct=4
Compiling query plan for /home/878352/Session-Management-Analyser/Flask_Queries/Flask-login-is-used-check/flask_library_used_check.ql.
[1/1] Found in cache: /home/878352/Session-Management-Analyser/Flask_Queries/Flask-login-is-used-check/flask_library_used_check.ql.
Starting evaluation of asterius27/codeql-extra-queries-python/Flask_Queries/Flask-login-is-used-check/flask_library_used_check.ql.
[1/1 eval 1m56s] Evaluation done; writing results to /home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/repositories/Django/thread4/HumanBrainProject_hbp_neuromorphic_platform/HumanBrainProject-ebrains-neuromorphic-job-queue-api-3c0e1f1-results/flask_library_used_check.bqrs.
Shutting down query evaluator.
Compiling query plan for /home/878352/Session-Management-Analyser/Django_Queries/Django-auth-is-used-check/django_library_used_check.ql.
[1/1] Found in cache: /home/878352/Session-Management-Analyser/Django_Queries/Django-auth-is-used-check/django_library_used_check.ql.
Starting evaluation of asterius27/codeql-extra-queries-python/Django_Queries/Django-auth-is-used-check/django_library_used_check.ql.
[1/1 eval 14.4s] Evaluation done; writing results to /home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/repositories/Django/thread4/HumanBrainProject_hbp_neuromorphic_platform/HumanBrainProject-ebrains-neuromorphic-job-queue-api-3c0e1f1-results/django_library_used_check.bqrs.
Shutting down query evaluator.
file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/python-single-analysis.js:97
        throw new Error("None of the supported libraries/frameworks is used");
              ^

Error: None of the supported libraries/frameworks is used
    at pythonAnalysis (file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/python-single-analysis.js:97:15)
    at file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/app.js:57:5
    at ModuleJob.run (node:internal/modules/esm/module_job:217:25)
    at async ModuleLoader.import (node:internal/modules/esm/loader:316:24)
    at async loadESM (node:internal/process/esm_loader:34:7)
    at async handleMainPromise (node:internal/modules/run_main:66:12)

Node.js v20.8.0

Time taken to run the queries on ./repositories/Django/thread4/HumanBrainProject_hbp_neuromorphic_platform/HumanBrainProject-ebrains-neuromorphic-job-queue-api-3c0e1f1: 270.477 seconds.
Time taken to run the queries on ./repositories/Django/thread4/INK-USC_AlpacaTag/INK-USC-AlpacaTag-ce04045: 2426.646 seconds.
Time taken to run the queries on ./repositories/Django/thread4/IT-Support-L2_mytek/IT-Support-L2-mytek-b168b39: 1718.081 seconds.
Time taken to run the queries on ./repositories/Django/thread4/JacobsAlumni_MemberManagement/JacobsAlumni-MemberManagement-2e06a7e: 2041.107 seconds.
Time taken to run the queries on ./repositories/Django/thread4/KimSoungRyoul_backend-handson-django/KimSoungRyoul-backend-handson-django-9547e9c: 2408.06 seconds.
Time taken to run the queries on ./repositories/Django/thread4/Komal7209_Farm_Fresh/Komal7209-Farm_Fresh-22c17a1: 2012.826 seconds.
Analysis failed for: ./repositories/Django/thread4/LegolasVzla_django-google-maps/LegolasVzla-django-google-maps-62efd2f Reason: Error: Command failed: npm start -- -s=./repositories/Django/thread4/LegolasVzla_django-google-maps/LegolasVzla-django-google-maps-62efd2f -l=python -t=11 -ct=4
Compiling query plan for /home/878352/Session-Management-Analyser/Flask_Queries/Flask-login-is-used-check/flask_library_used_check.ql.
[1/1] Found in cache: /home/878352/Session-Management-Analyser/Flask_Queries/Flask-login-is-used-check/flask_library_used_check.ql.
Starting evaluation of asterius27/codeql-extra-queries-python/Flask_Queries/Flask-login-is-used-check/flask_library_used_check.ql.
[1/1 eval 1m26s] Evaluation done; writing results to /home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/repositories/Django/thread4/LegolasVzla_django-google-maps/LegolasVzla-django-google-maps-62efd2f-results/flask_library_used_check.bqrs.
Shutting down query evaluator.
Compiling query plan for /home/878352/Session-Management-Analyser/Django_Queries/Django-auth-is-used-check/django_library_used_check.ql.
[1/1] Found in cache: /home/878352/Session-Management-Analyser/Django_Queries/Django-auth-is-used-check/django_library_used_check.ql.
Starting evaluation of asterius27/codeql-extra-queries-python/Django_Queries/Django-auth-is-used-check/django_library_used_check.ql.
[1/1 eval 13s] Evaluation done; writing results to /home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/repositories/Django/thread4/LegolasVzla_django-google-maps/LegolasVzla-django-google-maps-62efd2f-results/django_library_used_check.bqrs.
Shutting down query evaluator.
file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/python-single-analysis.js:97
        throw new Error("None of the supported libraries/frameworks is used");
              ^

Error: None of the supported libraries/frameworks is used
    at pythonAnalysis (file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/python-single-analysis.js:97:15)
    at file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/app.js:57:5
    at ModuleJob.run (node:internal/modules/esm/module_job:217:25)
    at async ModuleLoader.import (node:internal/modules/esm/loader:316:24)
    at async loadESM (node:internal/process/esm_loader:34:7)
    at async handleMainPromise (node:internal/modules/run_main:66:12)

Node.js v20.8.0

Time taken to run the queries on ./repositories/Django/thread4/LegolasVzla_django-google-maps/LegolasVzla-django-google-maps-62efd2f: 242.861 seconds.
Analysis failed for: ./repositories/Django/thread4/LordGhostX_QuickMed/LordGhostX-QuickMed-c5525a9 Reason: Error: Command failed: npm start -- -s=./repositories/Django/thread4/LordGhostX_QuickMed/LordGhostX-QuickMed-c5525a9 -l=python -t=11 -ct=4
Compiling query plan for /home/878352/Session-Management-Analyser/Flask_Queries/Flask-login-is-used-check/flask_library_used_check.ql.
[1/1] Found in cache: /home/878352/Session-Management-Analyser/Flask_Queries/Flask-login-is-used-check/flask_library_used_check.ql.
Starting evaluation of asterius27/codeql-extra-queries-python/Flask_Queries/Flask-login-is-used-check/flask_library_used_check.ql.
[1/1 eval 2m18s] Evaluation done; writing results to /home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/repositories/Django/thread4/LordGhostX_QuickMed/LordGhostX-QuickMed-c5525a9-results/flask_library_used_check.bqrs.
Shutting down query evaluator.
Compiling query plan for /home/878352/Session-Management-Analyser/Django_Queries/Django-auth-is-used-check/django_library_used_check.ql.
[1/1] Found in cache: /home/878352/Session-Management-Analyser/Django_Queries/Django-auth-is-used-check/django_library_used_check.ql.
Starting evaluation of asterius27/codeql-extra-queries-python/Django_Queries/Django-auth-is-used-check/django_library_used_check.ql.
[1/1 eval 23.6s] Evaluation done; writing results to /home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/repositories/Django/thread4/LordGhostX_QuickMed/LordGhostX-QuickMed-c5525a9-results/django_library_used_check.bqrs.
Shutting down query evaluator.
file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/python-single-analysis.js:97
        throw new Error("None of the supported libraries/frameworks is used");
              ^

Error: None of the supported libraries/frameworks is used
    at pythonAnalysis (file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/python-single-analysis.js:97:15)
    at file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/app.js:57:5
    at ModuleJob.run (node:internal/modules/esm/module_job:217:25)
    at async ModuleLoader.import (node:internal/modules/esm/loader:316:24)
    at async loadESM (node:internal/process/esm_loader:34:7)
    at async handleMainPromise (node:internal/modules/run_main:66:12)

Node.js v20.8.0

Time taken to run the queries on ./repositories/Django/thread4/LordGhostX_QuickMed/LordGhostX-QuickMed-c5525a9: 343.096 seconds.
Time taken to run the queries on ./repositories/Django/thread4/MapStory_mapstory/MapStory-mapstory-c305097: 2730.742 seconds.
Time taken to run the queries on ./repositories/Django/thread4/Marchhill_exhibition-inference/Marchhill-exhibition-inference-2e349da: 2831.114 seconds.
Time taken to run the queries on ./repositories/Django/thread4/NgoQuocBao1010_Shipping-System-Management/NgoQuocBao1010-Shipping-System-Management-a4aca6e: 2061.859 seconds.
Time taken to run the queries on ./repositories/Django/thread4/OurJSEditor_OurJSEditor/OurJSEditor-OurJSEditor-84f3e4b: 2003.525 seconds.
Time taken to run the queries on ./repositories/Django/thread4/Peagah-Vieira_Django-CRM-Web-Application/Peagah-Vieira-Django-CRM-Web-Application-2965490: 1967.903 seconds.
Analysis failed for: ./repositories/Django/thread4/SAML-Toolkits_python3-saml/SAML-Toolkits-python3-saml-a1211a8 Reason: Error: Command failed: npm start -- -s=./repositories/Django/thread4/SAML-Toolkits_python3-saml/SAML-Toolkits-python3-saml-a1211a8 -l=python -t=11 -ct=4
Compiling query plan for /home/878352/Session-Management-Analyser/Flask_Queries/Flask-login-is-used-check/flask_library_used_check.ql.
[1/1] Found in cache: /home/878352/Session-Management-Analyser/Flask_Queries/Flask-login-is-used-check/flask_library_used_check.ql.
Starting evaluation of asterius27/codeql-extra-queries-python/Flask_Queries/Flask-login-is-used-check/flask_library_used_check.ql.
[1/1 eval 1m18s] Evaluation done; writing results to /home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/repositories/Django/thread4/SAML-Toolkits_python3-saml/SAML-Toolkits-python3-saml-a1211a8-results/flask_library_used_check.bqrs.
Shutting down query evaluator.
Compiling query plan for /home/878352/Session-Management-Analyser/Django_Queries/Django-auth-is-used-check/django_library_used_check.ql.
[1/1] Found in cache: /home/878352/Session-Management-Analyser/Django_Queries/Django-auth-is-used-check/django_library_used_check.ql.
Starting evaluation of asterius27/codeql-extra-queries-python/Django_Queries/Django-auth-is-used-check/django_library_used_check.ql.
[1/1 eval 9.2s] Evaluation done; writing results to /home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/repositories/Django/thread4/SAML-Toolkits_python3-saml/SAML-Toolkits-python3-saml-a1211a8-results/django_library_used_check.bqrs.
Shutting down query evaluator.
file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/python-single-analysis.js:97
        throw new Error("None of the supported libraries/frameworks is used");
              ^

Error: None of the supported libraries/frameworks is used
    at pythonAnalysis (file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/python-single-analysis.js:97:15)
    at file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/app.js:57:5
    at ModuleJob.run (node:internal/modules/esm/module_job:217:25)
    at async ModuleLoader.import (node:internal/modules/esm/loader:316:24)
    at async loadESM (node:internal/process/esm_loader:34:7)
    at async handleMainPromise (node:internal/modules/run_main:66:12)

Node.js v20.8.0

Time taken to run the queries on ./repositories/Django/thread4/SAML-Toolkits_python3-saml/SAML-Toolkits-python3-saml-a1211a8: 218.775 seconds.
Time taken to run the queries on ./repositories/Django/thread4/SebastianRokholt_Full-Stack-Cloud-Dev-Capstone-Project/SebastianRokholt-Full-Stack-Cloud-Dev-Capstone-Project-ab0ddca: 2307.338 seconds.
Time taken to run the queries on ./repositories/Django/thread4/SethShickluna_COSC-4P02-R8Scholar/SethShickluna-COSC-4P02-R8Scholar-0d2d296: 1931.223 seconds.
Time taken to run the queries on ./repositories/Django/thread4/SwyftG_PeekpaCom/SwyftG-PeekpaCom-d90f85c: 1975.961 seconds.
Time taken to run the queries on ./repositories/Django/thread4/TACC_Core-Portal/TACC-Core-Portal-5a745e5: 2381.955 seconds.
Analysis failed for: ./repositories/Django/thread4/WHOIGit_whoi-hab-hub/WHOIGit-whoi-hab-hub-3646ac2 Reason: Error: Command failed: npm start -- -s=./repositories/Django/thread4/WHOIGit_whoi-hab-hub/WHOIGit-whoi-hab-hub-3646ac2 -l=python -t=11 -ct=4
Compiling query plan for /home/878352/Session-Management-Analyser/Flask_Queries/Flask-login-is-used-check/flask_library_used_check.ql.
[1/1] Found in cache: /home/878352/Session-Management-Analyser/Flask_Queries/Flask-login-is-used-check/flask_library_used_check.ql.
Starting evaluation of asterius27/codeql-extra-queries-python/Flask_Queries/Flask-login-is-used-check/flask_library_used_check.ql.
[1/1 eval 1m40s] Evaluation done; writing results to /home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/repositories/Django/thread4/WHOIGit_whoi-hab-hub/WHOIGit-whoi-hab-hub-3646ac2-results/flask_library_used_check.bqrs.
Shutting down query evaluator.
Compiling query plan for /home/878352/Session-Management-Analyser/Django_Queries/Django-auth-is-used-check/django_library_used_check.ql.
[1/1] Found in cache: /home/878352/Session-Management-Analyser/Django_Queries/Django-auth-is-used-check/django_library_used_check.ql.
Starting evaluation of asterius27/codeql-extra-queries-python/Django_Queries/Django-auth-is-used-check/django_library_used_check.ql.
[1/1 eval 16.3s] Evaluation done; writing results to /home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/repositories/Django/thread4/WHOIGit_whoi-hab-hub/WHOIGit-whoi-hab-hub-3646ac2-results/django_library_used_check.bqrs.
Shutting down query evaluator.
file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/python-single-analysis.js:97
        throw new Error("None of the supported libraries/frameworks is used");
              ^

Error: None of the supported libraries/frameworks is used
    at pythonAnalysis (file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/python-single-analysis.js:97:15)
    at file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/app.js:57:5
    at ModuleJob.run (node:internal/modules/esm/module_job:217:25)
    at async ModuleLoader.import (node:internal/modules/esm/loader:316:24)
    at async loadESM (node:internal/process/esm_loader:34:7)
    at async handleMainPromise (node:internal/modules/run_main:66:12)

Node.js v20.8.0

Time taken to run the queries on ./repositories/Django/thread4/WHOIGit_whoi-hab-hub/WHOIGit-whoi-hab-hub-3646ac2: 266.744 seconds.
Time taken to run the queries on ./repositories/Django/thread4/Williano_Bona-Blog/Williano-Bona-Blog-aa68f68: 2001.919 seconds.
Analysis failed for: ./repositories/Django/thread4/WordPress_openverse/WordPress-openverse-c50797a Reason: Error: Command failed: npm start -- -s=./repositories/Django/thread4/WordPress_openverse/WordPress-openverse-c50797a -l=python -t=11 -ct=4
Compiling query plan for /home/878352/Session-Management-Analyser/Flask_Queries/Flask-login-is-used-check/flask_library_used_check.ql.
[1/1] Found in cache: /home/878352/Session-Management-Analyser/Flask_Queries/Flask-login-is-used-check/flask_library_used_check.ql.
Starting evaluation of asterius27/codeql-extra-queries-python/Flask_Queries/Flask-login-is-used-check/flask_library_used_check.ql.
[1/1 eval 3m7s] Evaluation done; writing results to /home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/repositories/Django/thread4/WordPress_openverse/WordPress-openverse-c50797a-results/flask_library_used_check.bqrs.
Shutting down query evaluator.
Compiling query plan for /home/878352/Session-Management-Analyser/Django_Queries/Django-auth-is-used-check/django_library_used_check.ql.
[1/1] Found in cache: /home/878352/Session-Management-Analyser/Django_Queries/Django-auth-is-used-check/django_library_used_check.ql.
Starting evaluation of asterius27/codeql-extra-queries-python/Django_Queries/Django-auth-is-used-check/django_library_used_check.ql.
[1/1 eval 23s] Evaluation done; writing results to /home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/repositories/Django/thread4/WordPress_openverse/WordPress-openverse-c50797a-results/django_library_used_check.bqrs.
Shutting down query evaluator.
file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/python-single-analysis.js:97
        throw new Error("None of the supported libraries/frameworks is used");
              ^

Error: None of the supported libraries/frameworks is used
    at pythonAnalysis (file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/python-single-analysis.js:97:15)
    at file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/app.js:57:5
    at ModuleJob.run (node:internal/modules/esm/module_job:217:25)
    at async ModuleLoader.import (node:internal/modules/esm/loader:316:24)
    at async loadESM (node:internal/process/esm_loader:34:7)
    at async handleMainPromise (node:internal/modules/run_main:66:12)

Node.js v20.8.0

Time taken to run the queries on ./repositories/Django/thread4/WordPress_openverse/WordPress-openverse-c50797a: 397.948 seconds.
Time taken to run the queries on ./repositories/Django/thread4/Yawan-1_StackOverFlow--Clone/Yawan-1-StackOverFlow--Clone-a3ce476: 2066.635 seconds.
Time taken to run the queries on ./repositories/Django/thread4/Zarathustra2_TheShareGame/Zarathustra2-TheShareGame-50308e7: 2146.205 seconds.
Analysis failed for: ./repositories/Django/thread4/ZeroCoolHacker_easy-school/ZeroCoolHacker-easy-school-d55dc85 Reason: Error: Command failed: npm start -- -s=./repositories/Django/thread4/ZeroCoolHacker_easy-school/ZeroCoolHacker-easy-school-d55dc85 -l=python -t=11 -ct=4
Compiling query plan for /home/878352/Session-Management-Analyser/Flask_Queries/Flask-login-is-used-check/flask_library_used_check.ql.
[1/1] Found in cache: /home/878352/Session-Management-Analyser/Flask_Queries/Flask-login-is-used-check/flask_library_used_check.ql.
Starting evaluation of asterius27/codeql-extra-queries-python/Flask_Queries/Flask-login-is-used-check/flask_library_used_check.ql.
[1/1 eval 54.3s] Evaluation done; writing results to /home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/repositories/Django/thread4/ZeroCoolHacker_easy-school/ZeroCoolHacker-easy-school-d55dc85-results/flask_library_used_check.bqrs.
Shutting down query evaluator.
Compiling query plan for /home/878352/Session-Management-Analyser/Django_Queries/Django-auth-is-used-check/django_library_used_check.ql.
[1/1] Found in cache: /home/878352/Session-Management-Analyser/Django_Queries/Django-auth-is-used-check/django_library_used_check.ql.
Starting evaluation of asterius27/codeql-extra-queries-python/Django_Queries/Django-auth-is-used-check/django_library_used_check.ql.
[1/1 eval 14.2s] Evaluation done; writing results to /home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/repositories/Django/thread4/ZeroCoolHacker_easy-school/ZeroCoolHacker-easy-school-d55dc85-results/django_library_used_check.bqrs.
Shutting down query evaluator.
file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/python-single-analysis.js:97
        throw new Error("None of the supported libraries/frameworks is used");
              ^

Error: None of the supported libraries/frameworks is used
    at pythonAnalysis (file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/python-single-analysis.js:97:15)
    at file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/app.js:57:5
    at ModuleJob.run (node:internal/modules/esm/module_job:217:25)
    at async ModuleLoader.import (node:internal/modules/esm/loader:316:24)
    at async loadESM (node:internal/process/esm_loader:34:7)
    at async handleMainPromise (node:internal/modules/run_main:66:12)

Node.js v20.8.0

Time taken to run the queries on ./repositories/Django/thread4/ZeroCoolHacker_easy-school/ZeroCoolHacker-easy-school-d55dc85: 188.393 seconds.
Analysis failed for: ./repositories/Django/thread4/abdurraufraihan_django-chat/abdurraufraihan-django-chat-4a2a062 Reason: Error: Command failed: npm start -- -s=./repositories/Django/thread4/abdurraufraihan_django-chat/abdurraufraihan-django-chat-4a2a062 -l=python -t=11 -ct=4
Compiling query plan for /home/878352/Session-Management-Analyser/Flask_Queries/Flask-login-is-used-check/flask_library_used_check.ql.
[1/1] Found in cache: /home/878352/Session-Management-Analyser/Flask_Queries/Flask-login-is-used-check/flask_library_used_check.ql.
Starting evaluation of asterius27/codeql-extra-queries-python/Flask_Queries/Flask-login-is-used-check/flask_library_used_check.ql.
[1/1 eval 59.7s] Evaluation done; writing results to /home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/repositories/Django/thread4/abdurraufraihan_django-chat/abdurraufraihan-django-chat-4a2a062-results/flask_library_used_check.bqrs.
Shutting down query evaluator.
Compiling query plan for /home/878352/Session-Management-Analyser/Django_Queries/Django-auth-is-used-check/django_library_used_check.ql.
[1/1] Found in cache: /home/878352/Session-Management-Analyser/Django_Queries/Django-auth-is-used-check/django_library_used_check.ql.
Starting evaluation of asterius27/codeql-extra-queries-python/Django_Queries/Django-auth-is-used-check/django_library_used_check.ql.
[1/1 eval 10.1s] Evaluation done; writing results to /home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/repositories/Django/thread4/abdurraufraihan_django-chat/abdurraufraihan-django-chat-4a2a062-results/django_library_used_check.bqrs.
Shutting down query evaluator.
file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/python-single-analysis.js:97
        throw new Error("None of the supported libraries/frameworks is used");
              ^

Error: None of the supported libraries/frameworks is used
    at pythonAnalysis (file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/python-single-analysis.js:97:15)
    at file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/app.js:57:5
    at ModuleJob.run (node:internal/modules/esm/module_job:217:25)
    at async ModuleLoader.import (node:internal/modules/esm/loader:316:24)
    at async loadESM (node:internal/process/esm_loader:34:7)
    at async handleMainPromise (node:internal/modules/run_main:66:12)

Node.js v20.8.0

Time taken to run the queries on ./repositories/Django/thread4/abdurraufraihan_django-chat/abdurraufraihan-django-chat-4a2a062: 196.597 seconds.
Time taken to run the queries on ./repositories/Django/thread4/adeyosemanputra_pygoat/adeyosemanputra-pygoat-80c7370: 1997.577 seconds.
Time taken to run the queries on ./repositories/Django/thread4/ahmetbicer_react-native-delivery-app/ahmetbicer-react-native-delivery-app-f012386: 1817.8 seconds.
Analysis failed for: ./repositories/Django/thread4/altuntasmuhammet_eksisozluk-scraper/altuntasmuhammet-eksisozluk-scraper-8ba0c2a Reason: Error: Command failed: npm start -- -s=./repositories/Django/thread4/altuntasmuhammet_eksisozluk-scraper/altuntasmuhammet-eksisozluk-scraper-8ba0c2a -l=python -t=11 -ct=4
Compiling query plan for /home/878352/Session-Management-Analyser/Flask_Queries/Flask-login-is-used-check/flask_library_used_check.ql.
[1/1] Found in cache: /home/878352/Session-Management-Analyser/Flask_Queries/Flask-login-is-used-check/flask_library_used_check.ql.
Starting evaluation of asterius27/codeql-extra-queries-python/Flask_Queries/Flask-login-is-used-check/flask_library_used_check.ql.
[1/1 eval 1m42s] Evaluation done; writing results to /home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/repositories/Django/thread4/altuntasmuhammet_eksisozluk-scraper/altuntasmuhammet-eksisozluk-scraper-8ba0c2a-results/flask_library_used_check.bqrs.
Shutting down query evaluator.
Compiling query plan for /home/878352/Session-Management-Analyser/Django_Queries/Django-auth-is-used-check/django_library_used_check.ql.
[1/1] Found in cache: /home/878352/Session-Management-Analyser/Django_Queries/Django-auth-is-used-check/django_library_used_check.ql.
Starting evaluation of asterius27/codeql-extra-queries-python/Django_Queries/Django-auth-is-used-check/django_library_used_check.ql.
[1/1 eval 13.2s] Evaluation done; writing results to /home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/repositories/Django/thread4/altuntasmuhammet_eksisozluk-scraper/altuntasmuhammet-eksisozluk-scraper-8ba0c2a-results/django_library_used_check.bqrs.
Shutting down query evaluator.
file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/python-single-analysis.js:97
        throw new Error("None of the supported libraries/frameworks is used");
              ^

Error: None of the supported libraries/frameworks is used
    at pythonAnalysis (file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/python-single-analysis.js:97:15)
    at file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/app.js:57:5
    at ModuleJob.run (node:internal/modules/esm/module_job:217:25)
    at async ModuleLoader.import (node:internal/modules/esm/loader:316:24)
    at async loadESM (node:internal/process/esm_loader:34:7)
    at async handleMainPromise (node:internal/modules/run_main:66:12)

Node.js v20.8.0

Time taken to run the queries on ./repositories/Django/thread4/altuntasmuhammet_eksisozluk-scraper/altuntasmuhammet-eksisozluk-scraper-8ba0c2a: 263.02 seconds.
Time taken to run the queries on ./repositories/Django/thread4/andrenerd_django-multifactor-authentication/andrenerd-django-multifactor-authentication-4a8b94e: 2006.694 seconds.
Time taken to run the queries on ./repositories/Django/thread4/apragacz_django-rest-registration/apragacz-django-rest-registration-2b491bd: 2059.335 seconds.
Analysis failed for: ./repositories/Django/thread4/arthist-lmu_artigo/arthist-lmu-artigo-831d8a6 Reason: Error: Command failed: npm start -- -s=./repositories/Django/thread4/arthist-lmu_artigo/arthist-lmu-artigo-831d8a6 -l=python -t=11 -ct=4
Compiling query plan for /home/878352/Session-Management-Analyser/Flask_Queries/Flask-login-is-used-check/flask_library_used_check.ql.
[1/1] Found in cache: /home/878352/Session-Management-Analyser/Flask_Queries/Flask-login-is-used-check/flask_library_used_check.ql.
Starting evaluation of asterius27/codeql-extra-queries-python/Flask_Queries/Flask-login-is-used-check/flask_library_used_check.ql.
[1/1 eval 2m40s] Evaluation done; writing results to /home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/repositories/Django/thread4/arthist-lmu_artigo/arthist-lmu-artigo-831d8a6-results/flask_library_used_check.bqrs.
Shutting down query evaluator.
Compiling query plan for /home/878352/Session-Management-Analyser/Django_Queries/Django-auth-is-used-check/django_library_used_check.ql.
[1/1] Found in cache: /home/878352/Session-Management-Analyser/Django_Queries/Django-auth-is-used-check/django_library_used_check.ql.
Starting evaluation of asterius27/codeql-extra-queries-python/Django_Queries/Django-auth-is-used-check/django_library_used_check.ql.
[1/1 eval 24s] Evaluation done; writing results to /home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/repositories/Django/thread4/arthist-lmu_artigo/arthist-lmu-artigo-831d8a6-results/django_library_used_check.bqrs.
Shutting down query evaluator.
file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/python-single-analysis.js:97
        throw new Error("None of the supported libraries/frameworks is used");
              ^

Error: None of the supported libraries/frameworks is used
    at pythonAnalysis (file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/python-single-analysis.js:97:15)
    at file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/app.js:57:5
    at ModuleJob.run (node:internal/modules/esm/module_job:217:25)
    at async ModuleLoader.import (node:internal/modules/esm/loader:316:24)
    at async loadESM (node:internal/process/esm_loader:34:7)
    at async handleMainPromise (node:internal/modules/run_main:66:12)

Node.js v20.8.0

Time taken to run the queries on ./repositories/Django/thread4/arthist-lmu_artigo/arthist-lmu-artigo-831d8a6: 355.047 seconds.
Analysis failed for: ./repositories/Django/thread4/atharvabhide_DevFinder/atharvabhide-DevFinder-2c6ee74 Reason: Error: Command failed: npm start -- -s=./repositories/Django/thread4/atharvabhide_DevFinder/atharvabhide-DevFinder-2c6ee74 -l=python -t=11 -ct=4
Compiling query plan for /home/878352/Session-Management-Analyser/Flask_Queries/Flask-login-is-used-check/flask_library_used_check.ql.
[1/1] Found in cache: /home/878352/Session-Management-Analyser/Flask_Queries/Flask-login-is-used-check/flask_library_used_check.ql.
Starting evaluation of asterius27/codeql-extra-queries-python/Flask_Queries/Flask-login-is-used-check/flask_library_used_check.ql.
[1/1 eval 1m23s] Evaluation done; writing results to /home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/repositories/Django/thread4/atharvabhide_DevFinder/atharvabhide-DevFinder-2c6ee74-results/flask_library_used_check.bqrs.
Shutting down query evaluator.
Compiling query plan for /home/878352/Session-Management-Analyser/Django_Queries/Django-auth-is-used-check/django_library_used_check.ql.
[1/1] Found in cache: /home/878352/Session-Management-Analyser/Django_Queries/Django-auth-is-used-check/django_library_used_check.ql.
Starting evaluation of asterius27/codeql-extra-queries-python/Django_Queries/Django-auth-is-used-check/django_library_used_check.ql.
[1/1 eval 14s] Evaluation done; writing results to /home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/repositories/Django/thread4/atharvabhide_DevFinder/atharvabhide-DevFinder-2c6ee74-results/django_library_used_check.bqrs.
Shutting down query evaluator.
file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/python-single-analysis.js:97
        throw new Error("None of the supported libraries/frameworks is used");
              ^

Error: None of the supported libraries/frameworks is used
    at pythonAnalysis (file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/python-single-analysis.js:97:15)
    at file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/app.js:57:5
    at ModuleJob.run (node:internal/modules/esm/module_job:217:25)
    at async ModuleLoader.import (node:internal/modules/esm/loader:316:24)
    at async loadESM (node:internal/process/esm_loader:34:7)
    at async handleMainPromise (node:internal/modules/run_main:66:12)

Node.js v20.8.0

Time taken to run the queries on ./repositories/Django/thread4/atharvabhide_DevFinder/atharvabhide-DevFinder-2c6ee74: 222.471 seconds.
Time taken to run the queries on ./repositories/Django/thread4/avallbona_Impostor/avallbona-Impostor-9cd1bc7: 2111.523 seconds.
Time taken to run the queries on ./repositories/Django/thread4/b1ackc4t_14Finger/b1ackc4t-14Finger-2e29ff2: 2484.439 seconds.
Time taken to run the queries on ./repositories/Django/thread4/bcgov_gwells/bcgov-gwells-cec3e1c: 2594.643 seconds.
Time taken to run the queries on ./repositories/Django/thread4/bnbong_Project-Pinned/bnbong-Project-Pinned-44e6fea: 2199.638 seconds.
Analysis failed for: ./repositories/Django/thread4/bootc_netbox-chart/bootc-netbox-chart-12370a3 Reason: Error: Command failed: codeql database create ./repositories/Django/thread4/bootc_netbox-chart/bootc-netbox-chart-12370a3-database --language=python --source-root ./repositories/Django/thread4/bootc_netbox-chart/bootc-netbox-chart-12370a3 --threads=11
Initializing database at /home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/repositories/Django/thread4/bootc_netbox-chart/bootc-netbox-chart-12370a3-database.
Running build command: []
Finalizing database at /home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/repositories/Django/thread4/bootc_netbox-chart/bootc-netbox-chart-12370a3-database.
CodeQL did not detect any code written in languages supported by CodeQL. This can occur if the specified build commands failed to compile or process any code.
 - Confirm that there is some source code for the specified language in the project.
 - For codebases written in Go, JavaScript, TypeScript, and Python, do not specify 
   an explicit --command.
 - For other languages, the --command must specify a "clean" build which compiles 
   all the source code files without reusing existing build artefacts.

Time taken to run the queries on ./repositories/Django/thread4/bootc_netbox-chart/bootc-netbox-chart-12370a3: 13.94 seconds.
Analysis failed for: ./repositories/Django/thread4/cbaggini_baresoil/cbaggini-baresoil-ac720d5 Reason: Error: Command failed: npm start -- -s=./repositories/Django/thread4/cbaggini_baresoil/cbaggini-baresoil-ac720d5 -l=python -t=11 -ct=4
Compiling query plan for /home/878352/Session-Management-Analyser/Flask_Queries/Flask-login-is-used-check/flask_library_used_check.ql.
[1/1] Found in cache: /home/878352/Session-Management-Analyser/Flask_Queries/Flask-login-is-used-check/flask_library_used_check.ql.
Starting evaluation of asterius27/codeql-extra-queries-python/Flask_Queries/Flask-login-is-used-check/flask_library_used_check.ql.
[1/1 eval 1m14s] Evaluation done; writing results to /home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/repositories/Django/thread4/cbaggini_baresoil/cbaggini-baresoil-ac720d5-results/flask_library_used_check.bqrs.
Shutting down query evaluator.
Compiling query plan for /home/878352/Session-Management-Analyser/Django_Queries/Django-auth-is-used-check/django_library_used_check.ql.
[1/1] Found in cache: /home/878352/Session-Management-Analyser/Django_Queries/Django-auth-is-used-check/django_library_used_check.ql.
Starting evaluation of asterius27/codeql-extra-queries-python/Django_Queries/Django-auth-is-used-check/django_library_used_check.ql.
[1/1 eval 8.9s] Evaluation done; writing results to /home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/repositories/Django/thread4/cbaggini_baresoil/cbaggini-baresoil-ac720d5-results/django_library_used_check.bqrs.
Shutting down query evaluator.
file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/python-single-analysis.js:97
        throw new Error("None of the supported libraries/frameworks is used");
              ^

Error: None of the supported libraries/frameworks is used
    at pythonAnalysis (file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/python-single-analysis.js:97:15)
    at file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/app.js:57:5
    at ModuleJob.run (node:internal/modules/esm/module_job:217:25)
    at async ModuleLoader.import (node:internal/modules/esm/loader:316:24)
    at async loadESM (node:internal/process/esm_loader:34:7)
    at async handleMainPromise (node:internal/modules/run_main:66:12)

Node.js v20.8.0

Time taken to run the queries on ./repositories/Django/thread4/cbaggini_baresoil/cbaggini-baresoil-ac720d5: 220.546 seconds.
Time taken to run the queries on ./repositories/Django/thread4/cfe-lab_Kive/cfe-lab-Kive-c1c0c1a: 2650.671 seconds.
Time taken to run the queries on ./repositories/Django/thread4/citizenlabsgr_mittens/citizenlabsgr-mittens-e29da0c: 2044.509 seconds.
Analysis failed for: ./repositories/Django/thread4/civictechindex_CTI-website-backend/civictechindex-CTI-website-backend-c117ed4 Reason: Error: Command failed: npm start -- -s=./repositories/Django/thread4/civictechindex_CTI-website-backend/civictechindex-CTI-website-backend-c117ed4 -l=python -t=11 -ct=4
Compiling query plan for /home/878352/Session-Management-Analyser/Flask_Queries/Flask-login-is-used-check/flask_library_used_check.ql.
[1/1] Found in cache: /home/878352/Session-Management-Analyser/Flask_Queries/Flask-login-is-used-check/flask_library_used_check.ql.
Starting evaluation of asterius27/codeql-extra-queries-python/Flask_Queries/Flask-login-is-used-check/flask_library_used_check.ql.
[1/1 eval 1m24s] Evaluation done; writing results to /home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/repositories/Django/thread4/civictechindex_CTI-website-backend/civictechindex-CTI-website-backend-c117ed4-results/flask_library_used_check.bqrs.
Shutting down query evaluator.
Compiling query plan for /home/878352/Session-Management-Analyser/Django_Queries/Django-auth-is-used-check/django_library_used_check.ql.
[1/1] Found in cache: /home/878352/Session-Management-Analyser/Django_Queries/Django-auth-is-used-check/django_library_used_check.ql.
Starting evaluation of asterius27/codeql-extra-queries-python/Django_Queries/Django-auth-is-used-check/django_library_used_check.ql.
[1/1 eval 11.7s] Evaluation done; writing results to /home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/repositories/Django/thread4/civictechindex_CTI-website-backend/civictechindex-CTI-website-backend-c117ed4-results/django_library_used_check.bqrs.
Shutting down query evaluator.
file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/python-single-analysis.js:97
        throw new Error("None of the supported libraries/frameworks is used");
              ^

Error: None of the supported libraries/frameworks is used
    at pythonAnalysis (file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/python-single-analysis.js:97:15)
    at file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/app.js:57:5
    at ModuleJob.run (node:internal/modules/esm/module_job:217:25)
    at async ModuleLoader.import (node:internal/modules/esm/loader:316:24)
    at async loadESM (node:internal/process/esm_loader:34:7)
    at async handleMainPromise (node:internal/modules/run_main:66:12)

Node.js v20.8.0

Time taken to run the queries on ./repositories/Django/thread4/civictechindex_CTI-website-backend/civictechindex-CTI-website-backend-c117ed4: 216.582 seconds.
Time taken to run the queries on ./repositories/Django/thread4/cknoll_django-sober/cknoll-django-sober-124dbde: 1715.408 seconds.
Time taken to run the queries on ./repositories/Django/thread4/cnr-ibba_shiny-server/cnr-ibba-shiny-server-9762902: 1672.47 seconds.
Time taken to run the queries on ./repositories/Django/thread4/codefisher_djangopress/codefisher-djangopress-ab51f44: 1732.716 seconds.
Analysis failed for: ./repositories/Django/thread4/codingforentrepreneurs_Serverless-Django/codingforentrepreneurs-Serverless-Django-77908ab Reason: Error: Command failed: npm start -- -s=./repositories/Django/thread4/codingforentrepreneurs_Serverless-Django/codingforentrepreneurs-Serverless-Django-77908ab -l=python -t=11 -ct=4
Compiling query plan for /home/878352/Session-Management-Analyser/Flask_Queries/Flask-login-is-used-check/flask_library_used_check.ql.
[1/1] Found in cache: /home/878352/Session-Management-Analyser/Flask_Queries/Flask-login-is-used-check/flask_library_used_check.ql.
Starting evaluation of asterius27/codeql-extra-queries-python/Flask_Queries/Flask-login-is-used-check/flask_library_used_check.ql.
[1/1 eval 1m1s] Evaluation done; writing results to /home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/repositories/Django/thread4/codingforentrepreneurs_Serverless-Django/codingforentrepreneurs-Serverless-Django-77908ab-results/flask_library_used_check.bqrs.
Shutting down query evaluator.
Compiling query plan for /home/878352/Session-Management-Analyser/Django_Queries/Django-auth-is-used-check/django_library_used_check.ql.
[1/1] Found in cache: /home/878352/Session-Management-Analyser/Django_Queries/Django-auth-is-used-check/django_library_used_check.ql.
Starting evaluation of asterius27/codeql-extra-queries-python/Django_Queries/Django-auth-is-used-check/django_library_used_check.ql.
[1/1 eval 7.4s] Evaluation done; writing results to /home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/repositories/Django/thread4/codingforentrepreneurs_Serverless-Django/codingforentrepreneurs-Serverless-Django-77908ab-results/django_library_used_check.bqrs.
Shutting down query evaluator.
file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/python-single-analysis.js:97
        throw new Error("None of the supported libraries/frameworks is used");
              ^

Error: None of the supported libraries/frameworks is used
    at pythonAnalysis (file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/python-single-analysis.js:97:15)
    at file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/app.js:57:5
    at ModuleJob.run (node:internal/modules/esm/module_job:217:25)
    at async ModuleLoader.import (node:internal/modules/esm/loader:316:24)
    at async loadESM (node:internal/process/esm_loader:34:7)
    at async handleMainPromise (node:internal/modules/run_main:66:12)

Node.js v20.8.0

Time taken to run the queries on ./repositories/Django/thread4/codingforentrepreneurs_Serverless-Django/codingforentrepreneurs-Serverless-Django-77908ab: 187.966 seconds.
Time taken to run the queries on ./repositories/Django/thread4/colinsullivan_bingo-board/colinsullivan-bingo-board-4c75e38: 1610.664 seconds.
Analysis failed for: ./repositories/Django/thread4/consbio_ncdjango/consbio-ncdjango-cb800e2 Reason: Error: Command failed: npm start -- -s=./repositories/Django/thread4/consbio_ncdjango/consbio-ncdjango-cb800e2 -l=python -t=11 -ct=4
Compiling query plan for /home/878352/Session-Management-Analyser/Flask_Queries/Flask-login-is-used-check/flask_library_used_check.ql.
[1/1] Found in cache: /home/878352/Session-Management-Analyser/Flask_Queries/Flask-login-is-used-check/flask_library_used_check.ql.
Starting evaluation of asterius27/codeql-extra-queries-python/Flask_Queries/Flask-login-is-used-check/flask_library_used_check.ql.
[1/1 eval 2m7s] Evaluation done; writing results to /home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/repositories/Django/thread4/consbio_ncdjango/consbio-ncdjango-cb800e2-results/flask_library_used_check.bqrs.
Shutting down query evaluator.
Compiling query plan for /home/878352/Session-Management-Analyser/Django_Queries/Django-auth-is-used-check/django_library_used_check.ql.
[1/1] Found in cache: /home/878352/Session-Management-Analyser/Django_Queries/Django-auth-is-used-check/django_library_used_check.ql.
Starting evaluation of asterius27/codeql-extra-queries-python/Django_Queries/Django-auth-is-used-check/django_library_used_check.ql.
[1/1 eval 18.5s] Evaluation done; writing results to /home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/repositories/Django/thread4/consbio_ncdjango/consbio-ncdjango-cb800e2-results/django_library_used_check.bqrs.
Shutting down query evaluator.
file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/python-single-analysis.js:97
        throw new Error("None of the supported libraries/frameworks is used");
              ^

Error: None of the supported libraries/frameworks is used
    at pythonAnalysis (file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/python-single-analysis.js:97:15)
    at file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/app.js:57:5
    at ModuleJob.run (node:internal/modules/esm/module_job:217:25)
    at async ModuleLoader.import (node:internal/modules/esm/loader:316:24)
    at async loadESM (node:internal/process/esm_loader:34:7)
    at async handleMainPromise (node:internal/modules/run_main:66:12)

Node.js v20.8.0

Time taken to run the queries on ./repositories/Django/thread4/consbio_ncdjango/consbio-ncdjango-cb800e2: 303.305 seconds.
Analysis failed for: ./repositories/Django/thread4/cookiecutter_cookiecutter-django/cookiecutter-cookiecutter-django-6dcf48f Reason: Error: Command failed: npm start -- -s=./repositories/Django/thread4/cookiecutter_cookiecutter-django/cookiecutter-cookiecutter-django-6dcf48f -l=python -t=11 -ct=4
Compiling query plan for /home/878352/Session-Management-Analyser/Flask_Queries/Flask-login-is-used-check/flask_library_used_check.ql.
[1/1] Found in cache: /home/878352/Session-Management-Analyser/Flask_Queries/Flask-login-is-used-check/flask_library_used_check.ql.
Starting evaluation of asterius27/codeql-extra-queries-python/Flask_Queries/Flask-login-is-used-check/flask_library_used_check.ql.
[1/1 eval 1m32s] Evaluation done; writing results to /home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/repositories/Django/thread4/cookiecutter_cookiecutter-django/cookiecutter-cookiecutter-django-6dcf48f-results/flask_library_used_check.bqrs.
Shutting down query evaluator.
Compiling query plan for /home/878352/Session-Management-Analyser/Django_Queries/Django-auth-is-used-check/django_library_used_check.ql.
[1/1] Found in cache: /home/878352/Session-Management-Analyser/Django_Queries/Django-auth-is-used-check/django_library_used_check.ql.
Starting evaluation of asterius27/codeql-extra-queries-python/Django_Queries/Django-auth-is-used-check/django_library_used_check.ql.
[1/1 eval 15.1s] Evaluation done; writing results to /home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/repositories/Django/thread4/cookiecutter_cookiecutter-django/cookiecutter-cookiecutter-django-6dcf48f-results/django_library_used_check.bqrs.
Shutting down query evaluator.
file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/python-single-analysis.js:97
        throw new Error("None of the supported libraries/frameworks is used");
              ^

Error: None of the supported libraries/frameworks is used
    at pythonAnalysis (file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/python-single-analysis.js:97:15)
    at file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/app.js:57:5
    at ModuleJob.run (node:internal/modules/esm/module_job:217:25)
    at async ModuleLoader.import (node:internal/modules/esm/loader:316:24)
    at async loadESM (node:internal/process/esm_loader:34:7)
    at async handleMainPromise (node:internal/modules/run_main:66:12)

Node.js v20.8.0

Time taken to run the queries on ./repositories/Django/thread4/cookiecutter_cookiecutter-django/cookiecutter-cookiecutter-django-6dcf48f: 244.851 seconds.
Analysis failed for: ./repositories/Django/thread4/dandi_dandi-archive/dandi-dandi-archive-5b6c66a Reason: Error: Command failed: npm start -- -s=./repositories/Django/thread4/dandi_dandi-archive/dandi-dandi-archive-5b6c66a -l=python -t=11 -ct=4
Compiling query plan for /home/878352/Session-Management-Analyser/Flask_Queries/Flask-login-is-used-check/flask_library_used_check.ql.
[1/1] Found in cache: /home/878352/Session-Management-Analyser/Flask_Queries/Flask-login-is-used-check/flask_library_used_check.ql.
Starting evaluation of asterius27/codeql-extra-queries-python/Flask_Queries/Flask-login-is-used-check/flask_library_used_check.ql.
[1/1 eval 1m33s] Evaluation done; writing results to /home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/repositories/Django/thread4/dandi_dandi-archive/dandi-dandi-archive-5b6c66a-results/flask_library_used_check.bqrs.
Shutting down query evaluator.
Compiling query plan for /home/878352/Session-Management-Analyser/Django_Queries/Django-auth-is-used-check/django_library_used_check.ql.
[1/1] Found in cache: /home/878352/Session-Management-Analyser/Django_Queries/Django-auth-is-used-check/django_library_used_check.ql.
Starting evaluation of asterius27/codeql-extra-queries-python/Django_Queries/Django-auth-is-used-check/django_library_used_check.ql.
[1/1 eval 15.6s] Evaluation done; writing results to /home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/repositories/Django/thread4/dandi_dandi-archive/dandi-dandi-archive-5b6c66a-results/django_library_used_check.bqrs.
Shutting down query evaluator.
file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/python-single-analysis.js:97
        throw new Error("None of the supported libraries/frameworks is used");
              ^

Error: None of the supported libraries/frameworks is used
    at pythonAnalysis (file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/python-single-analysis.js:97:15)
    at file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/app.js:57:5
    at ModuleJob.run (node:internal/modules/esm/module_job:217:25)
    at async ModuleLoader.import (node:internal/modules/esm/loader:316:24)
    at async loadESM (node:internal/process/esm_loader:34:7)
    at async handleMainPromise (node:internal/modules/run_main:66:12)

Node.js v20.8.0

Time taken to run the queries on ./repositories/Django/thread4/dandi_dandi-archive/dandi-dandi-archive-5b6c66a: 247.277 seconds.
Time taken to run the queries on ./repositories/Django/thread4/daxslab_comohay/daxslab-comohay-2036054: 2530.434 seconds.
Time taken to run the queries on ./repositories/Django/thread4/devenes_my-aws-capstone-project/devenes-my-aws-capstone-project-f0787ef: 1772.414 seconds.
Time taken to run the queries on ./repositories/Django/thread4/dhruv-goyal-10_Linkedin-Clone-backend/dhruv-goyal-10-Linkedin-Clone-backend-913b5ba: 2297.781 seconds.
Time taken to run the queries on ./repositories/Django/thread4/dianajoan_e-learning_app/dianajoan-e-learning_app-d488812: 1724.106 seconds.
Time taken to run the queries on ./repositories/Django/thread4/dixitaayush8_ReviewTunez/dixitaayush8-ReviewTunez-19b6528: 1963.036 seconds.
Time taken to run the queries on ./repositories/Django/thread4/diyajaiswal11_Bloggitt/diyajaiswal11-Bloggitt-aca9335: 1786.239 seconds.
Time taken to run the queries on ./repositories/Django/thread4/e-m-b-a_embark/e-m-b-a-embark-6c17ddb: 1748.554 seconds.
Analysis failed for: ./repositories/Django/thread4/edelvalle_reactor/edelvalle-reactor-20bbe7d Reason: Error: Command failed: npm start -- -s=./repositories/Django/thread4/edelvalle_reactor/edelvalle-reactor-20bbe7d -l=python -t=11 -ct=4
Compiling query plan for /home/878352/Session-Management-Analyser/Flask_Queries/Flask-login-is-used-check/flask_library_used_check.ql.
[1/1] Found in cache: /home/878352/Session-Management-Analyser/Flask_Queries/Flask-login-is-used-check/flask_library_used_check.ql.
Starting evaluation of asterius27/codeql-extra-queries-python/Flask_Queries/Flask-login-is-used-check/flask_library_used_check.ql.
[1/1 eval 1m11s] Evaluation done; writing results to /home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/repositories/Django/thread4/edelvalle_reactor/edelvalle-reactor-20bbe7d-results/flask_library_used_check.bqrs.
Shutting down query evaluator.
Compiling query plan for /home/878352/Session-Management-Analyser/Django_Queries/Django-auth-is-used-check/django_library_used_check.ql.
[1/1] Found in cache: /home/878352/Session-Management-Analyser/Django_Queries/Django-auth-is-used-check/django_library_used_check.ql.
Starting evaluation of asterius27/codeql-extra-queries-python/Django_Queries/Django-auth-is-used-check/django_library_used_check.ql.
[1/1 eval 6.4s] Evaluation done; writing results to /home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/repositories/Django/thread4/edelvalle_reactor/edelvalle-reactor-20bbe7d-results/django_library_used_check.bqrs.
Shutting down query evaluator.
file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/python-single-analysis.js:97
        throw new Error("None of the supported libraries/frameworks is used");
              ^

Error: None of the supported libraries/frameworks is used
    at pythonAnalysis (file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/python-single-analysis.js:97:15)
    at file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/app.js:57:5
    at ModuleJob.run (node:internal/modules/esm/module_job:217:25)
    at async ModuleLoader.import (node:internal/modules/esm/loader:316:24)
    at async loadESM (node:internal/process/esm_loader:34:7)
    at async handleMainPromise (node:internal/modules/run_main:66:12)

Node.js v20.8.0

Time taken to run the queries on ./repositories/Django/thread4/edelvalle_reactor/edelvalle-reactor-20bbe7d: 187.603 seconds.
Time taken to run the queries on ./repositories/Django/thread4/edwilson543_timetable-solutions/edwilson543-timetable-solutions-9280431: 2383.21 seconds.
Time taken to run the queries on ./repositories/Django/thread4/elixir-luxembourg_daisy/elixir-luxembourg-daisy-5b3f164: 2416.576 seconds.
Analysis failed for: ./repositories/Django/thread4/evolutio_djavue3/evolutio-djavue3-5dc1d61 Reason: Error: Command failed: npm start -- -s=./repositories/Django/thread4/evolutio_djavue3/evolutio-djavue3-5dc1d61 -l=python -t=11 -ct=4
Compiling query plan for /home/878352/Session-Management-Analyser/Flask_Queries/Flask-login-is-used-check/flask_library_used_check.ql.
[1/1] Found in cache: /home/878352/Session-Management-Analyser/Flask_Queries/Flask-login-is-used-check/flask_library_used_check.ql.
Starting evaluation of asterius27/codeql-extra-queries-python/Flask_Queries/Flask-login-is-used-check/flask_library_used_check.ql.
[1/1 eval 57.3s] Evaluation done; writing results to /home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/repositories/Django/thread4/evolutio_djavue3/evolutio-djavue3-5dc1d61-results/flask_library_used_check.bqrs.
Shutting down query evaluator.
Compiling query plan for /home/878352/Session-Management-Analyser/Django_Queries/Django-auth-is-used-check/django_library_used_check.ql.
[1/1] Found in cache: /home/878352/Session-Management-Analyser/Django_Queries/Django-auth-is-used-check/django_library_used_check.ql.
Starting evaluation of asterius27/codeql-extra-queries-python/Django_Queries/Django-auth-is-used-check/django_library_used_check.ql.
[1/1 eval 12.3s] Evaluation done; writing results to /home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/repositories/Django/thread4/evolutio_djavue3/evolutio-djavue3-5dc1d61-results/django_library_used_check.bqrs.
Shutting down query evaluator.
file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/python-single-analysis.js:97
        throw new Error("None of the supported libraries/frameworks is used");
              ^

Error: None of the supported libraries/frameworks is used
    at pythonAnalysis (file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/python-single-analysis.js:97:15)
    at file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/app.js:57:5
    at ModuleJob.run (node:internal/modules/esm/module_job:217:25)
    at async ModuleLoader.import (node:internal/modules/esm/loader:316:24)
    at async loadESM (node:internal/process/esm_loader:34:7)
    at async handleMainPromise (node:internal/modules/run_main:66:12)

Node.js v20.8.0

Time taken to run the queries on ./repositories/Django/thread4/evolutio_djavue3/evolutio-djavue3-5dc1d61: 190.429 seconds.
Time taken to run the queries on ./repositories/Django/thread4/fengjinqi_website/fengjinqi-website-340eece: 2122.654 seconds.
Analysis failed for: ./repositories/Django/thread4/fga-eps-mds_2021.2-INDICAA/fga-eps-mds-2021.2-INDICAA-4fac9b9 Reason: Error: Command failed: npm start -- -s=./repositories/Django/thread4/fga-eps-mds_2021.2-INDICAA/fga-eps-mds-2021.2-INDICAA-4fac9b9 -l=python -t=11 -ct=4
Compiling query plan for /home/878352/Session-Management-Analyser/Flask_Queries/Flask-login-is-used-check/flask_library_used_check.ql.
[1/1] Found in cache: /home/878352/Session-Management-Analyser/Flask_Queries/Flask-login-is-used-check/flask_library_used_check.ql.
Starting evaluation of asterius27/codeql-extra-queries-python/Flask_Queries/Flask-login-is-used-check/flask_library_used_check.ql.
[1/1 eval 1m5s] Evaluation done; writing results to /home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/repositories/Django/thread4/fga-eps-mds_2021.2-INDICAA/fga-eps-mds-2021.2-INDICAA-4fac9b9-results/flask_library_used_check.bqrs.
Shutting down query evaluator.
Compiling query plan for /home/878352/Session-Management-Analyser/Django_Queries/Django-auth-is-used-check/django_library_used_check.ql.
[1/1] Found in cache: /home/878352/Session-Management-Analyser/Django_Queries/Django-auth-is-used-check/django_library_used_check.ql.
Starting evaluation of asterius27/codeql-extra-queries-python/Django_Queries/Django-auth-is-used-check/django_library_used_check.ql.
[1/1 eval 9.5s] Evaluation done; writing results to /home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/repositories/Django/thread4/fga-eps-mds_2021.2-INDICAA/fga-eps-mds-2021.2-INDICAA-4fac9b9-results/django_library_used_check.bqrs.
Shutting down query evaluator.
file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/python-single-analysis.js:97
        throw new Error("None of the supported libraries/frameworks is used");
              ^

Error: None of the supported libraries/frameworks is used
    at pythonAnalysis (file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/python-single-analysis.js:97:15)
    at file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/app.js:57:5
    at ModuleJob.run (node:internal/modules/esm/module_job:217:25)
    at async ModuleLoader.import (node:internal/modules/esm/loader:316:24)
    at async loadESM (node:internal/process/esm_loader:34:7)
    at async handleMainPromise (node:internal/modules/run_main:66:12)

Node.js v20.8.0

Time taken to run the queries on ./repositories/Django/thread4/fga-eps-mds_2021.2-INDICAA/fga-eps-mds-2021.2-INDICAA-4fac9b9: 181.659 seconds.
Analysis failed for: ./repositories/Django/thread4/foo290_Django-Verify-Email/foo290-Django-Verify-Email-725e77a Reason: Error: Command failed: npm start -- -s=./repositories/Django/thread4/foo290_Django-Verify-Email/foo290-Django-Verify-Email-725e77a -l=python -t=11 -ct=4
Compiling query plan for /home/878352/Session-Management-Analyser/Flask_Queries/Flask-login-is-used-check/flask_library_used_check.ql.
[1/1] Found in cache: /home/878352/Session-Management-Analyser/Flask_Queries/Flask-login-is-used-check/flask_library_used_check.ql.
Starting evaluation of asterius27/codeql-extra-queries-python/Flask_Queries/Flask-login-is-used-check/flask_library_used_check.ql.
[1/1 eval 1m8s] Evaluation done; writing results to /home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/repositories/Django/thread4/foo290_Django-Verify-Email/foo290-Django-Verify-Email-725e77a-results/flask_library_used_check.bqrs.
Shutting down query evaluator.
Compiling query plan for /home/878352/Session-Management-Analyser/Django_Queries/Django-auth-is-used-check/django_library_used_check.ql.
[1/1] Found in cache: /home/878352/Session-Management-Analyser/Django_Queries/Django-auth-is-used-check/django_library_used_check.ql.
Starting evaluation of asterius27/codeql-extra-queries-python/Django_Queries/Django-auth-is-used-check/django_library_used_check.ql.
[1/1 eval 9.2s] Evaluation done; writing results to /home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/repositories/Django/thread4/foo290_Django-Verify-Email/foo290-Django-Verify-Email-725e77a-results/django_library_used_check.bqrs.
Shutting down query evaluator.
file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/python-single-analysis.js:97
        throw new Error("None of the supported libraries/frameworks is used");
              ^

Error: None of the supported libraries/frameworks is used
    at pythonAnalysis (file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/python-single-analysis.js:97:15)
    at file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/app.js:57:5
    at ModuleJob.run (node:internal/modules/esm/module_job:217:25)
    at async ModuleLoader.import (node:internal/modules/esm/loader:316:24)
    at async loadESM (node:internal/process/esm_loader:34:7)
    at async handleMainPromise (node:internal/modules/run_main:66:12)

Node.js v20.8.0

Time taken to run the queries on ./repositories/Django/thread4/foo290_Django-Verify-Email/foo290-Django-Verify-Email-725e77a: 198.583 seconds.
Time taken to run the queries on ./repositories/Django/thread4/genuine-oj_backend/genuine-oj-backend-23dcd19: 2062.739 seconds.
Time taken to run the queries on ./repositories/Django/thread4/gibran-abdillah_chat-app/gibran-abdillah-chat-app-c189966: 1767.727 seconds.
Analysis failed for: ./repositories/Django/thread4/gmega_callblocker/gmega-callblocker-212ca85 Reason: Error: Command failed: npm start -- -s=./repositories/Django/thread4/gmega_callblocker/gmega-callblocker-212ca85 -l=python -t=11 -ct=4
Compiling query plan for /home/878352/Session-Management-Analyser/Flask_Queries/Flask-login-is-used-check/flask_library_used_check.ql.
[1/1] Found in cache: /home/878352/Session-Management-Analyser/Flask_Queries/Flask-login-is-used-check/flask_library_used_check.ql.
Starting evaluation of asterius27/codeql-extra-queries-python/Flask_Queries/Flask-login-is-used-check/flask_library_used_check.ql.
[1/1 eval 1m4s] Evaluation done; writing results to /home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/repositories/Django/thread4/gmega_callblocker/gmega-callblocker-212ca85-results/flask_library_used_check.bqrs.
Shutting down query evaluator.
Compiling query plan for /home/878352/Session-Management-Analyser/Django_Queries/Django-auth-is-used-check/django_library_used_check.ql.
[1/1] Found in cache: /home/878352/Session-Management-Analyser/Django_Queries/Django-auth-is-used-check/django_library_used_check.ql.
Starting evaluation of asterius27/codeql-extra-queries-python/Django_Queries/Django-auth-is-used-check/django_library_used_check.ql.
[1/1 eval 9.4s] Evaluation done; writing results to /home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/repositories/Django/thread4/gmega_callblocker/gmega-callblocker-212ca85-results/django_library_used_check.bqrs.
Shutting down query evaluator.
file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/python-single-analysis.js:97
        throw new Error("None of the supported libraries/frameworks is used");
              ^

Error: None of the supported libraries/frameworks is used
    at pythonAnalysis (file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/python-single-analysis.js:97:15)
    at file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/app.js:57:5
    at ModuleJob.run (node:internal/modules/esm/module_job:217:25)
    at async ModuleLoader.import (node:internal/modules/esm/loader:316:24)
    at async loadESM (node:internal/process/esm_loader:34:7)
    at async handleMainPromise (node:internal/modules/run_main:66:12)

Node.js v20.8.0

Time taken to run the queries on ./repositories/Django/thread4/gmega_callblocker/gmega-callblocker-212ca85: 192.519 seconds.
Time taken to run the queries on ./repositories/Django/thread4/goer3_OPMS_v3/goer3-OPMS_v3-219a671: 1927.434 seconds.
Time taken to run the queries on ./repositories/Django/thread4/gthole_gedgo/gthole-gedgo-a2561f4: 1800.117 seconds.
Time taken to run the queries on ./repositories/Django/thread4/hhyo_Archery/hhyo-Archery-bf06ae2: 2212.882 seconds.
Time taken to run the queries on ./repositories/Django/thread4/horilla-opensource_horilla/horilla-opensource-horilla-6375987: 1957.765 seconds.
Time taken to run the queries on ./repositories/Django/thread4/hsv-dot-beer_hsvdotbeer/hsv-dot-beer-hsvdotbeer-9b8f27d: 2081.643 seconds.
Time taken to run the queries on ./repositories/Django/thread4/iamnihal_warf/iamnihal-warf-2b59a8a: 2276.526 seconds.
Time taken to run the queries on ./repositories/Django/thread4/iiacoban42_DelfiTLM/iiacoban42-DelfiTLM-6bafe2f: 1955.539 seconds.
Time taken to run the queries on ./repositories/Django/thread4/itscontained_automarkwatched/itscontained-automarkwatched-bbe6bb3: 1852.063 seconds.
Time taken to run the queries on ./repositories/Django/thread4/jlaine_django-coconuts/jlaine-django-coconuts-9e5064f: 1648.255 seconds.
Analysis failed for: ./repositories/Django/thread4/laws-africa_indigo/laws-africa-indigo-be3d8f7 Reason: Error: Command failed: npm start -- -s=./repositories/Django/thread4/laws-africa_indigo/laws-africa-indigo-be3d8f7 -l=python -t=11 -ct=4
Compiling query plan for /home/878352/Session-Management-Analyser/Flask_Queries/Flask-login-is-used-check/flask_library_used_check.ql.
[1/1] Found in cache: /home/878352/Session-Management-Analyser/Flask_Queries/Flask-login-is-used-check/flask_library_used_check.ql.
Starting evaluation of asterius27/codeql-extra-queries-python/Flask_Queries/Flask-login-is-used-check/flask_library_used_check.ql.
[1/1 eval 1m38s] Evaluation done; writing results to /home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/repositories/Django/thread4/laws-africa_indigo/laws-africa-indigo-be3d8f7-results/flask_library_used_check.bqrs.
Shutting down query evaluator.
Compiling query plan for /home/878352/Session-Management-Analyser/Django_Queries/Django-auth-is-used-check/django_library_used_check.ql.
[1/1] Found in cache: /home/878352/Session-Management-Analyser/Django_Queries/Django-auth-is-used-check/django_library_used_check.ql.
Starting evaluation of asterius27/codeql-extra-queries-python/Django_Queries/Django-auth-is-used-check/django_library_used_check.ql.
[1/1 eval 12.6s] Evaluation done; writing results to /home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/repositories/Django/thread4/laws-africa_indigo/laws-africa-indigo-be3d8f7-results/django_library_used_check.bqrs.
Shutting down query evaluator.
file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/python-single-analysis.js:97
        throw new Error("None of the supported libraries/frameworks is used");
              ^

Error: None of the supported libraries/frameworks is used
    at pythonAnalysis (file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/python-single-analysis.js:97:15)
    at file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/app.js:57:5
    at ModuleJob.run (node:internal/modules/esm/module_job:217:25)
    at async ModuleLoader.import (node:internal/modules/esm/loader:316:24)
    at async loadESM (node:internal/process/esm_loader:34:7)
    at async handleMainPromise (node:internal/modules/run_main:66:12)

Node.js v20.8.0

Time taken to run the queries on ./repositories/Django/thread4/laws-africa_indigo/laws-africa-indigo-be3d8f7: 242.203 seconds.
Time taken to run the queries on ./repositories/Django/thread4/liqianglog_django-vue-admin/liqianglog-django-vue-admin-4257ca0: 1954.428 seconds.
Analysis failed for: ./repositories/Django/thread4/maranemil_howto/maranemil-howto-fe04fcc Reason: Error: Command failed: npm start -- -s=./repositories/Django/thread4/maranemil_howto/maranemil-howto-fe04fcc -l=python -t=11 -ct=4
Compiling query plan for /home/878352/Session-Management-Analyser/Flask_Queries/Flask-login-is-used-check/flask_library_used_check.ql.
[1/1] Found in cache: /home/878352/Session-Management-Analyser/Flask_Queries/Flask-login-is-used-check/flask_library_used_check.ql.
Starting evaluation of asterius27/codeql-extra-queries-python/Flask_Queries/Flask-login-is-used-check/flask_library_used_check.ql.
[1/1 eval 2m54s] Evaluation done; writing results to /home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/repositories/Django/thread4/maranemil_howto/maranemil-howto-fe04fcc-results/flask_library_used_check.bqrs.
Shutting down query evaluator.
Compiling query plan for /home/878352/Session-Management-Analyser/Django_Queries/Django-auth-is-used-check/django_library_used_check.ql.
[1/1] Found in cache: /home/878352/Session-Management-Analyser/Django_Queries/Django-auth-is-used-check/django_library_used_check.ql.
Starting evaluation of asterius27/codeql-extra-queries-python/Django_Queries/Django-auth-is-used-check/django_library_used_check.ql.
[1/1 eval 15.8s] Evaluation done; writing results to /home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/repositories/Django/thread4/maranemil_howto/maranemil-howto-fe04fcc-results/django_library_used_check.bqrs.
Shutting down query evaluator.
file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/python-single-analysis.js:97
        throw new Error("None of the supported libraries/frameworks is used");
              ^

Error: None of the supported libraries/frameworks is used
    at pythonAnalysis (file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/python-single-analysis.js:97:15)
    at file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/app.js:57:5
    at ModuleJob.run (node:internal/modules/esm/module_job:217:25)
    at async ModuleLoader.import (node:internal/modules/esm/loader:316:24)
    at async loadESM (node:internal/process/esm_loader:34:7)
    at async handleMainPromise (node:internal/modules/run_main:66:12)

Node.js v20.8.0

Time taken to run the queries on ./repositories/Django/thread4/maranemil_howto/maranemil-howto-fe04fcc: 372.145 seconds.
Time taken to run the queries on ./repositories/Django/thread4/marceloFA_deadlines_app/marceloFA-deadlines_app-1c412bf: 1665.409 seconds.
Analysis failed for: ./repositories/Django/thread4/mblayman_homeschool/mblayman-homeschool-75481fd Reason: Error: Command failed: npm start -- -s=./repositories/Django/thread4/mblayman_homeschool/mblayman-homeschool-75481fd -l=python -t=11 -ct=4
Compiling query plan for /home/878352/Session-Management-Analyser/Flask_Queries/Flask-login-is-used-check/flask_library_used_check.ql.
[1/1] Found in cache: /home/878352/Session-Management-Analyser/Flask_Queries/Flask-login-is-used-check/flask_library_used_check.ql.
Starting evaluation of asterius27/codeql-extra-queries-python/Flask_Queries/Flask-login-is-used-check/flask_library_used_check.ql.
[1/1 eval 1m18s] Evaluation done; writing results to /home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/repositories/Django/thread4/mblayman_homeschool/mblayman-homeschool-75481fd-results/flask_library_used_check.bqrs.
Shutting down query evaluator.
Compiling query plan for /home/878352/Session-Management-Analyser/Django_Queries/Django-auth-is-used-check/django_library_used_check.ql.
[1/1] Found in cache: /home/878352/Session-Management-Analyser/Django_Queries/Django-auth-is-used-check/django_library_used_check.ql.
Starting evaluation of asterius27/codeql-extra-queries-python/Django_Queries/Django-auth-is-used-check/django_library_used_check.ql.
[1/1 eval 16.1s] Evaluation done; writing results to /home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/repositories/Django/thread4/mblayman_homeschool/mblayman-homeschool-75481fd-results/django_library_used_check.bqrs.
Shutting down query evaluator.
file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/python-single-analysis.js:97
        throw new Error("None of the supported libraries/frameworks is used");
              ^

Error: None of the supported libraries/frameworks is used
    at pythonAnalysis (file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/python-single-analysis.js:97:15)
    at file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/app.js:57:5
    at ModuleJob.run (node:internal/modules/esm/module_job:217:25)
    at async ModuleLoader.import (node:internal/modules/esm/loader:316:24)
    at async loadESM (node:internal/process/esm_loader:34:7)
    at async handleMainPromise (node:internal/modules/run_main:66:12)

Node.js v20.8.0

Time taken to run the queries on ./repositories/Django/thread4/mblayman_homeschool/mblayman-homeschool-75481fd: 217.599 seconds.
Analysis failed for: ./repositories/Django/thread4/mdiener21_python-geospatial-analysis-cookbook/mdiener21-python-geospatial-analysis-cookbook-0c10a32 Reason: Error: Command failed: npm start -- -s=./repositories/Django/thread4/mdiener21_python-geospatial-analysis-cookbook/mdiener21-python-geospatial-analysis-cookbook-0c10a32 -l=python -t=11 -ct=4
Compiling query plan for /home/878352/Session-Management-Analyser/Flask_Queries/Flask-login-is-used-check/flask_library_used_check.ql.
[1/1] Found in cache: /home/878352/Session-Management-Analyser/Flask_Queries/Flask-login-is-used-check/flask_library_used_check.ql.
Starting evaluation of asterius27/codeql-extra-queries-python/Flask_Queries/Flask-login-is-used-check/flask_library_used_check.ql.
[1/1 eval 2m33s] Evaluation done; writing results to /home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/repositories/Django/thread4/mdiener21_python-geospatial-analysis-cookbook/mdiener21-python-geospatial-analysis-cookbook-0c10a32-results/flask_library_used_check.bqrs.
Shutting down query evaluator.
Compiling query plan for /home/878352/Session-Management-Analyser/Django_Queries/Django-auth-is-used-check/django_library_used_check.ql.
[1/1] Found in cache: /home/878352/Session-Management-Analyser/Django_Queries/Django-auth-is-used-check/django_library_used_check.ql.
Starting evaluation of asterius27/codeql-extra-queries-python/Django_Queries/Django-auth-is-used-check/django_library_used_check.ql.
[1/1 eval 15s] Evaluation done; writing results to /home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/repositories/Django/thread4/mdiener21_python-geospatial-analysis-cookbook/mdiener21-python-geospatial-analysis-cookbook-0c10a32-results/django_library_used_check.bqrs.
Shutting down query evaluator.
file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/python-single-analysis.js:97
        throw new Error("None of the supported libraries/frameworks is used");
              ^

Error: None of the supported libraries/frameworks is used
    at pythonAnalysis (file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/python-single-analysis.js:97:15)
    at file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/app.js:57:5
    at ModuleJob.run (node:internal/modules/esm/module_job:217:25)
    at async ModuleLoader.import (node:internal/modules/esm/loader:316:24)
    at async loadESM (node:internal/process/esm_loader:34:7)
    at async handleMainPromise (node:internal/modules/run_main:66:12)

Node.js v20.8.0

Time taken to run the queries on ./repositories/Django/thread4/mdiener21_python-geospatial-analysis-cookbook/mdiener21-python-geospatial-analysis-cookbook-0c10a32: 330.972 seconds.
Time taken to run the queries on ./repositories/Django/thread4/mdn_django-diy-blog/mdn-django-diy-blog-4af1515: 1673.466 seconds.
Analysis failed for: ./repositories/Django/thread4/melizalab_django-lab-inventory/melizalab-django-lab-inventory-f721675 Reason: Error: Command failed: npm start -- -s=./repositories/Django/thread4/melizalab_django-lab-inventory/melizalab-django-lab-inventory-f721675 -l=python -t=11 -ct=4
Compiling query plan for /home/878352/Session-Management-Analyser/Flask_Queries/Flask-login-is-used-check/flask_library_used_check.ql.
[1/1] Found in cache: /home/878352/Session-Management-Analyser/Flask_Queries/Flask-login-is-used-check/flask_library_used_check.ql.
Starting evaluation of asterius27/codeql-extra-queries-python/Flask_Queries/Flask-login-is-used-check/flask_library_used_check.ql.
[1/1 eval 58.8s] Evaluation done; writing results to /home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/repositories/Django/thread4/melizalab_django-lab-inventory/melizalab-django-lab-inventory-f721675-results/flask_library_used_check.bqrs.
Shutting down query evaluator.
Compiling query plan for /home/878352/Session-Management-Analyser/Django_Queries/Django-auth-is-used-check/django_library_used_check.ql.
[1/1] Found in cache: /home/878352/Session-Management-Analyser/Django_Queries/Django-auth-is-used-check/django_library_used_check.ql.
Starting evaluation of asterius27/codeql-extra-queries-python/Django_Queries/Django-auth-is-used-check/django_library_used_check.ql.
[1/1 eval 10.1s] Evaluation done; writing results to /home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/repositories/Django/thread4/melizalab_django-lab-inventory/melizalab-django-lab-inventory-f721675-results/django_library_used_check.bqrs.
Shutting down query evaluator.
file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/python-single-analysis.js:97
        throw new Error("None of the supported libraries/frameworks is used");
              ^

Error: None of the supported libraries/frameworks is used
    at pythonAnalysis (file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/python-single-analysis.js:97:15)
    at file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/app.js:57:5
    at ModuleJob.run (node:internal/modules/esm/module_job:217:25)
    at async ModuleLoader.import (node:internal/modules/esm/loader:316:24)
    at async loadESM (node:internal/process/esm_loader:34:7)
    at async handleMainPromise (node:internal/modules/run_main:66:12)

Node.js v20.8.0

Time taken to run the queries on ./repositories/Django/thread4/melizalab_django-lab-inventory/melizalab-django-lab-inventory-f721675: 186.806 seconds.
Time taken to run the queries on ./repositories/Django/thread4/metabolism-of-cities_metabolism-of-cities-platform/metabolism-of-cities-metabolism-of-cities-platform-d18d726: 2293.565 seconds.
Analysis failed for: ./repositories/Django/thread4/milymilo-ballpit_hoodie/milymilo-ballpit-hoodie-51fa1e9 Reason: Error: Command failed: npm start -- -s=./repositories/Django/thread4/milymilo-ballpit_hoodie/milymilo-ballpit-hoodie-51fa1e9 -l=python -t=11 -ct=4
Compiling query plan for /home/878352/Session-Management-Analyser/Flask_Queries/Flask-login-is-used-check/flask_library_used_check.ql.
[1/1] Found in cache: /home/878352/Session-Management-Analyser/Flask_Queries/Flask-login-is-used-check/flask_library_used_check.ql.
Starting evaluation of asterius27/codeql-extra-queries-python/Flask_Queries/Flask-login-is-used-check/flask_library_used_check.ql.
[1/1 eval 1m0s] Evaluation done; writing results to /home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/repositories/Django/thread4/milymilo-ballpit_hoodie/milymilo-ballpit-hoodie-51fa1e9-results/flask_library_used_check.bqrs.
Shutting down query evaluator.
Compiling query plan for /home/878352/Session-Management-Analyser/Django_Queries/Django-auth-is-used-check/django_library_used_check.ql.
[1/1] Found in cache: /home/878352/Session-Management-Analyser/Django_Queries/Django-auth-is-used-check/django_library_used_check.ql.
Starting evaluation of asterius27/codeql-extra-queries-python/Django_Queries/Django-auth-is-used-check/django_library_used_check.ql.
[1/1 eval 12.4s] Evaluation done; writing results to /home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/repositories/Django/thread4/milymilo-ballpit_hoodie/milymilo-ballpit-hoodie-51fa1e9-results/django_library_used_check.bqrs.
Shutting down query evaluator.
file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/python-single-analysis.js:97
        throw new Error("None of the supported libraries/frameworks is used");
              ^

Error: None of the supported libraries/frameworks is used
    at pythonAnalysis (file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/python-single-analysis.js:97:15)
    at file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/app.js:57:5
    at ModuleJob.run (node:internal/modules/esm/module_job:217:25)
    at async ModuleLoader.import (node:internal/modules/esm/loader:316:24)
    at async loadESM (node:internal/process/esm_loader:34:7)
    at async handleMainPromise (node:internal/modules/run_main:66:12)

Node.js v20.8.0

Time taken to run the queries on ./repositories/Django/thread4/milymilo-ballpit_hoodie/milymilo-ballpit-hoodie-51fa1e9: 185 seconds.
Analysis failed for: ./repositories/Django/thread4/moodbeat_back/moodbeat-back-6e0b78e Reason: Error: Command failed: npm start -- -s=./repositories/Django/thread4/moodbeat_back/moodbeat-back-6e0b78e -l=python -t=11 -ct=4
Compiling query plan for /home/878352/Session-Management-Analyser/Flask_Queries/Flask-login-is-used-check/flask_library_used_check.ql.
[1/1] Found in cache: /home/878352/Session-Management-Analyser/Flask_Queries/Flask-login-is-used-check/flask_library_used_check.ql.
Starting evaluation of asterius27/codeql-extra-queries-python/Flask_Queries/Flask-login-is-used-check/flask_library_used_check.ql.
[1/1 eval 1m9s] Evaluation done; writing results to /home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/repositories/Django/thread4/moodbeat_back/moodbeat-back-6e0b78e-results/flask_library_used_check.bqrs.
Shutting down query evaluator.
Compiling query plan for /home/878352/Session-Management-Analyser/Django_Queries/Django-auth-is-used-check/django_library_used_check.ql.
[1/1] Found in cache: /home/878352/Session-Management-Analyser/Django_Queries/Django-auth-is-used-check/django_library_used_check.ql.
Starting evaluation of asterius27/codeql-extra-queries-python/Django_Queries/Django-auth-is-used-check/django_library_used_check.ql.
[1/1 eval 11.6s] Evaluation done; writing results to /home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/repositories/Django/thread4/moodbeat_back/moodbeat-back-6e0b78e-results/django_library_used_check.bqrs.
Shutting down query evaluator.
file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/python-single-analysis.js:97
        throw new Error("None of the supported libraries/frameworks is used");
              ^

Error: None of the supported libraries/frameworks is used
    at pythonAnalysis (file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/python-single-analysis.js:97:15)
    at file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/app.js:57:5
    at ModuleJob.run (node:internal/modules/esm/module_job:217:25)
    at async ModuleLoader.import (node:internal/modules/esm/loader:316:24)
    at async loadESM (node:internal/process/esm_loader:34:7)
    at async handleMainPromise (node:internal/modules/run_main:66:12)

Node.js v20.8.0

Time taken to run the queries on ./repositories/Django/thread4/moodbeat_back/moodbeat-back-6e0b78e: 189.488 seconds.
Time taken to run the queries on ./repositories/Django/thread4/nc2U_rebs/nc2U-rebs-c3648a4: 1743.926 seconds.
Time taken to run the queries on ./repositories/Django/thread4/neeraj1909_quiz/neeraj1909-quiz-2885456: 1656.534 seconds.
Time taken to run the queries on ./repositories/Django/thread4/netbox-community_netbox/netbox-community-netbox-edb7d24: 4349.816 seconds.
Time taken to run the queries on ./repositories/Django/thread4/nikhilh-20_ELFEN/nikhilh-20-ELFEN-2cef508: 1768.82 seconds.
Time taken to run the queries on ./repositories/Django/thread4/nikitosikvn1_django-odms-project/nikitosikvn1-django-odms-project-1bde2ec: 1473.592 seconds.
Analysis failed for: ./repositories/Django/thread4/no-gravity_web_app_from_scratch/no-gravity-web_app_from_scratch-a182d66 Reason: Error: Command failed: codeql database create ./repositories/Django/thread4/no-gravity_web_app_from_scratch/no-gravity-web_app_from_scratch-a182d66-database --language=python --source-root ./repositories/Django/thread4/no-gravity_web_app_from_scratch/no-gravity-web_app_from_scratch-a182d66 --threads=11
Initializing database at /home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/repositories/Django/thread4/no-gravity_web_app_from_scratch/no-gravity-web_app_from_scratch-a182d66-database.
Running build command: []
Finalizing database at /home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/repositories/Django/thread4/no-gravity_web_app_from_scratch/no-gravity-web_app_from_scratch-a182d66-database.
CodeQL did not detect any code written in languages supported by CodeQL. This can occur if the specified build commands failed to compile or process any code.
 - Confirm that there is some source code for the specified language in the project.
 - For codebases written in Go, JavaScript, TypeScript, and Python, do not specify 
   an explicit --command.
 - For other languages, the --command must specify a "clean" build which compiles 
   all the source code files without reusing existing build artefacts.

Time taken to run the queries on ./repositories/Django/thread4/no-gravity_web_app_from_scratch/no-gravity-web_app_from_scratch-a182d66: 9.716 seconds.
Time taken to run the queries on ./repositories/Django/thread4/nyph-infosec_daggerboard/nyph-infosec-daggerboard-a215021: 1297.573 seconds.
Analysis failed for: ./repositories/Django/thread4/ojarva_file-manager-django/ojarva-file-manager-django-b8ed3e4 Reason: Error: Command failed: npm start -- -s=./repositories/Django/thread4/ojarva_file-manager-django/ojarva-file-manager-django-b8ed3e4 -l=python -t=11 -ct=4
Compiling query plan for /home/878352/Session-Management-Analyser/Flask_Queries/Flask-login-is-used-check/flask_library_used_check.ql.
[1/1] Found in cache: /home/878352/Session-Management-Analyser/Flask_Queries/Flask-login-is-used-check/flask_library_used_check.ql.
Starting evaluation of asterius27/codeql-extra-queries-python/Flask_Queries/Flask-login-is-used-check/flask_library_used_check.ql.
[1/1 eval 43.2s] Evaluation done; writing results to /home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/repositories/Django/thread4/ojarva_file-manager-django/ojarva-file-manager-django-b8ed3e4-results/flask_library_used_check.bqrs.
Shutting down query evaluator.
Compiling query plan for /home/878352/Session-Management-Analyser/Django_Queries/Django-auth-is-used-check/django_library_used_check.ql.
[1/1] Found in cache: /home/878352/Session-Management-Analyser/Django_Queries/Django-auth-is-used-check/django_library_used_check.ql.
Starting evaluation of asterius27/codeql-extra-queries-python/Django_Queries/Django-auth-is-used-check/django_library_used_check.ql.
[1/1 eval 6.7s] Evaluation done; writing results to /home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/repositories/Django/thread4/ojarva_file-manager-django/ojarva-file-manager-django-b8ed3e4-results/django_library_used_check.bqrs.
Shutting down query evaluator.
file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/python-single-analysis.js:97
        throw new Error("None of the supported libraries/frameworks is used");
              ^

Error: None of the supported libraries/frameworks is used
    at pythonAnalysis (file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/python-single-analysis.js:97:15)
    at file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/app.js:57:5
    at ModuleJob.run (node:internal/modules/esm/module_job:217:25)
    at async ModuleLoader.import (node:internal/modules/esm/loader:316:24)
    at async loadESM (node:internal/process/esm_loader:34:7)
    at async handleMainPromise (node:internal/modules/run_main:66:12)

Node.js v20.8.0

Time taken to run the queries on ./repositories/Django/thread4/ojarva_file-manager-django/ojarva-file-manager-django-b8ed3e4: 129.031 seconds.
Analysis failed for: ./repositories/Django/thread4/okfde_fragdenstaat_de/okfde-fragdenstaat_de-798e4cb Reason: Error: Command failed: npm start -- -s=./repositories/Django/thread4/okfde_fragdenstaat_de/okfde-fragdenstaat_de-798e4cb -l=python -t=11 -ct=4
Compiling query plan for /home/878352/Session-Management-Analyser/Flask_Queries/Flask-login-is-used-check/flask_library_used_check.ql.
[1/1] Found in cache: /home/878352/Session-Management-Analyser/Flask_Queries/Flask-login-is-used-check/flask_library_used_check.ql.
Starting evaluation of asterius27/codeql-extra-queries-python/Flask_Queries/Flask-login-is-used-check/flask_library_used_check.ql.
[1/1 eval 55.3s] Evaluation done; writing results to /home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/repositories/Django/thread4/okfde_fragdenstaat_de/okfde-fragdenstaat_de-798e4cb-results/flask_library_used_check.bqrs.
Shutting down query evaluator.
Compiling query plan for /home/878352/Session-Management-Analyser/Django_Queries/Django-auth-is-used-check/django_library_used_check.ql.
[1/1] Found in cache: /home/878352/Session-Management-Analyser/Django_Queries/Django-auth-is-used-check/django_library_used_check.ql.
Starting evaluation of asterius27/codeql-extra-queries-python/Django_Queries/Django-auth-is-used-check/django_library_used_check.ql.
[1/1 eval 12.5s] Evaluation done; writing results to /home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/repositories/Django/thread4/okfde_fragdenstaat_de/okfde-fragdenstaat_de-798e4cb-results/django_library_used_check.bqrs.
Shutting down query evaluator.
file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/python-single-analysis.js:97
        throw new Error("None of the supported libraries/frameworks is used");
              ^

Error: None of the supported libraries/frameworks is used
    at pythonAnalysis (file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/python-single-analysis.js:97:15)
    at file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/app.js:57:5
    at ModuleJob.run (node:internal/modules/esm/module_job:217:25)
    at async ModuleLoader.import (node:internal/modules/esm/loader:316:24)
    at async loadESM (node:internal/process/esm_loader:34:7)
    at async handleMainPromise (node:internal/modules/run_main:66:12)

Node.js v20.8.0

Time taken to run the queries on ./repositories/Django/thread4/okfde_fragdenstaat_de/okfde-fragdenstaat_de-798e4cb: 170.795 seconds.
Analysis failed for: ./repositories/Django/thread4/olopost_snotra_rss/olopost-snotra_rss-3829158 Reason: Error: Command failed: npm start -- -s=./repositories/Django/thread4/olopost_snotra_rss/olopost-snotra_rss-3829158 -l=python -t=11 -ct=4
Compiling query plan for /home/878352/Session-Management-Analyser/Flask_Queries/Flask-login-is-used-check/flask_library_used_check.ql.
[1/1] Found in cache: /home/878352/Session-Management-Analyser/Flask_Queries/Flask-login-is-used-check/flask_library_used_check.ql.
Starting evaluation of asterius27/codeql-extra-queries-python/Flask_Queries/Flask-login-is-used-check/flask_library_used_check.ql.
[1/1 eval 43.2s] Evaluation done; writing results to /home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/repositories/Django/thread4/olopost_snotra_rss/olopost-snotra_rss-3829158-results/flask_library_used_check.bqrs.
Shutting down query evaluator.
Compiling query plan for /home/878352/Session-Management-Analyser/Django_Queries/Django-auth-is-used-check/django_library_used_check.ql.
[1/1] Found in cache: /home/878352/Session-Management-Analyser/Django_Queries/Django-auth-is-used-check/django_library_used_check.ql.
Starting evaluation of asterius27/codeql-extra-queries-python/Django_Queries/Django-auth-is-used-check/django_library_used_check.ql.
[1/1 eval 7.5s] Evaluation done; writing results to /home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/repositories/Django/thread4/olopost_snotra_rss/olopost-snotra_rss-3829158-results/django_library_used_check.bqrs.
Shutting down query evaluator.
file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/python-single-analysis.js:97
        throw new Error("None of the supported libraries/frameworks is used");
              ^

Error: None of the supported libraries/frameworks is used
    at pythonAnalysis (file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/python-single-analysis.js:97:15)
    at file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/app.js:57:5
    at ModuleJob.run (node:internal/modules/esm/module_job:217:25)
    at async ModuleLoader.import (node:internal/modules/esm/loader:316:24)
    at async loadESM (node:internal/process/esm_loader:34:7)
    at async handleMainPromise (node:internal/modules/run_main:66:12)

Node.js v20.8.0

Time taken to run the queries on ./repositories/Django/thread4/olopost_snotra_rss/olopost-snotra_rss-3829158: 126.397 seconds.
Analysis failed for: ./repositories/Django/thread4/open-risk_equinox/open-risk-equinox-10ba681 Reason: Error: Command failed: npm start -- -s=./repositories/Django/thread4/open-risk_equinox/open-risk-equinox-10ba681 -l=python -t=11 -ct=4
Compiling query plan for /home/878352/Session-Management-Analyser/Flask_Queries/Flask-login-is-used-check/flask_library_used_check.ql.
[1/1] Found in cache: /home/878352/Session-Management-Analyser/Flask_Queries/Flask-login-is-used-check/flask_library_used_check.ql.
Starting evaluation of asterius27/codeql-extra-queries-python/Flask_Queries/Flask-login-is-used-check/flask_library_used_check.ql.
[1/1 eval 1m54s] Evaluation done; writing results to /home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/repositories/Django/thread4/open-risk_equinox/open-risk-equinox-10ba681-results/flask_library_used_check.bqrs.
Shutting down query evaluator.
Compiling query plan for /home/878352/Session-Management-Analyser/Django_Queries/Django-auth-is-used-check/django_library_used_check.ql.
[1/1] Found in cache: /home/878352/Session-Management-Analyser/Django_Queries/Django-auth-is-used-check/django_library_used_check.ql.
Starting evaluation of asterius27/codeql-extra-queries-python/Django_Queries/Django-auth-is-used-check/django_library_used_check.ql.
[1/1 eval 18.3s] Evaluation done; writing results to /home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/repositories/Django/thread4/open-risk_equinox/open-risk-equinox-10ba681-results/django_library_used_check.bqrs.
Shutting down query evaluator.
file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/python-single-analysis.js:97
        throw new Error("None of the supported libraries/frameworks is used");
              ^

Error: None of the supported libraries/frameworks is used
    at pythonAnalysis (file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/python-single-analysis.js:97:15)
    at file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/app.js:57:5
    at ModuleJob.run (node:internal/modules/esm/module_job:217:25)
    at async ModuleLoader.import (node:internal/modules/esm/loader:316:24)
    at async loadESM (node:internal/process/esm_loader:34:7)
    at async handleMainPromise (node:internal/modules/run_main:66:12)

Node.js v20.8.0

Time taken to run the queries on ./repositories/Django/thread4/open-risk_equinox/open-risk-equinox-10ba681: 254.04 seconds.
Analysis failed for: ./repositories/Django/thread4/peterkuria_django-react-in-docker-microservices/peterkuria-django-react-in-docker-microservices-7d13787 Reason: Error: Command failed: npm start -- -s=./repositories/Django/thread4/peterkuria_django-react-in-docker-microservices/peterkuria-django-react-in-docker-microservices-7d13787 -l=python -t=11 -ct=4
Compiling query plan for /home/878352/Session-Management-Analyser/Flask_Queries/Flask-login-is-used-check/flask_library_used_check.ql.
[1/1] Found in cache: /home/878352/Session-Management-Analyser/Flask_Queries/Flask-login-is-used-check/flask_library_used_check.ql.
Starting evaluation of asterius27/codeql-extra-queries-python/Flask_Queries/Flask-login-is-used-check/flask_library_used_check.ql.
[1/1 eval 40.5s] Evaluation done; writing results to /home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/repositories/Django/thread4/peterkuria_django-react-in-docker-microservices/peterkuria-django-react-in-docker-microservices-7d13787-results/flask_library_used_check.bqrs.
Shutting down query evaluator.
Compiling query plan for /home/878352/Session-Management-Analyser/Django_Queries/Django-auth-is-used-check/django_library_used_check.ql.
[1/1] Found in cache: /home/878352/Session-Management-Analyser/Django_Queries/Django-auth-is-used-check/django_library_used_check.ql.
Starting evaluation of asterius27/codeql-extra-queries-python/Django_Queries/Django-auth-is-used-check/django_library_used_check.ql.
[1/1 eval 5.8s] Evaluation done; writing results to /home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/repositories/Django/thread4/peterkuria_django-react-in-docker-microservices/peterkuria-django-react-in-docker-microservices-7d13787-results/django_library_used_check.bqrs.
Shutting down query evaluator.
file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/python-single-analysis.js:97
        throw new Error("None of the supported libraries/frameworks is used");
              ^

Error: None of the supported libraries/frameworks is used
    at pythonAnalysis (file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/python-single-analysis.js:97:15)
    at file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/app.js:57:5
    at ModuleJob.run (node:internal/modules/esm/module_job:217:25)
    at async ModuleLoader.import (node:internal/modules/esm/loader:316:24)
    at async loadESM (node:internal/process/esm_loader:34:7)
    at async handleMainPromise (node:internal/modules/run_main:66:12)

Node.js v20.8.0

Time taken to run the queries on ./repositories/Django/thread4/peterkuria_django-react-in-docker-microservices/peterkuria-django-react-in-docker-microservices-7d13787: 129.223 seconds.
Analysis failed for: ./repositories/Django/thread4/pinax_pinax-teams/pinax-pinax-teams-054b2dd Reason: Error: Command failed: npm start -- -s=./repositories/Django/thread4/pinax_pinax-teams/pinax-pinax-teams-054b2dd -l=python -t=11 -ct=4
Compiling query plan for /home/878352/Session-Management-Analyser/Flask_Queries/Flask-login-is-used-check/flask_library_used_check.ql.
[1/1] Found in cache: /home/878352/Session-Management-Analyser/Flask_Queries/Flask-login-is-used-check/flask_library_used_check.ql.
Starting evaluation of asterius27/codeql-extra-queries-python/Flask_Queries/Flask-login-is-used-check/flask_library_used_check.ql.
[1/1 eval 42.5s] Evaluation done; writing results to /home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/repositories/Django/thread4/pinax_pinax-teams/pinax-pinax-teams-054b2dd-results/flask_library_used_check.bqrs.
Shutting down query evaluator.
Compiling query plan for /home/878352/Session-Management-Analyser/Django_Queries/Django-auth-is-used-check/django_library_used_check.ql.
[1/1] Found in cache: /home/878352/Session-Management-Analyser/Django_Queries/Django-auth-is-used-check/django_library_used_check.ql.
Starting evaluation of asterius27/codeql-extra-queries-python/Django_Queries/Django-auth-is-used-check/django_library_used_check.ql.
[1/1 eval 9.7s] Evaluation done; writing results to /home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/repositories/Django/thread4/pinax_pinax-teams/pinax-pinax-teams-054b2dd-results/django_library_used_check.bqrs.
Shutting down query evaluator.
file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/python-single-analysis.js:97
        throw new Error("None of the supported libraries/frameworks is used");
              ^

Error: None of the supported libraries/frameworks is used
    at pythonAnalysis (file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/python-single-analysis.js:97:15)
    at file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/app.js:57:5
    at ModuleJob.run (node:internal/modules/esm/module_job:217:25)
    at async ModuleLoader.import (node:internal/modules/esm/loader:316:24)
    at async loadESM (node:internal/process/esm_loader:34:7)
    at async handleMainPromise (node:internal/modules/run_main:66:12)

Node.js v20.8.0

Time taken to run the queries on ./repositories/Django/thread4/pinax_pinax-teams/pinax-pinax-teams-054b2dd: 120.743 seconds.
Time taken to run the queries on ./repositories/Django/thread4/progrmoiz_Saiki/progrmoiz-Saiki-dd792d1: 1140.296 seconds.
Time taken to run the queries on ./repositories/Django/thread4/quipucords_quipucords/quipucords-quipucords-33e2daa: 1398.531 seconds.
Analysis failed for: ./repositories/Django/thread4/raatmarien_my-qr.art/raatmarien-my-qr.art-26a4492 Reason: Error: Command failed: npm start -- -s=./repositories/Django/thread4/raatmarien_my-qr.art/raatmarien-my-qr.art-26a4492 -l=python -t=11 -ct=4
Compiling query plan for /home/878352/Session-Management-Analyser/Flask_Queries/Flask-login-is-used-check/flask_library_used_check.ql.
[1/1] Found in cache: /home/878352/Session-Management-Analyser/Flask_Queries/Flask-login-is-used-check/flask_library_used_check.ql.
Starting evaluation of asterius27/codeql-extra-queries-python/Flask_Queries/Flask-login-is-used-check/flask_library_used_check.ql.
[1/1 eval 36.8s] Evaluation done; writing results to /home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/repositories/Django/thread4/raatmarien_my-qr.art/raatmarien-my-qr.art-26a4492-results/flask_library_used_check.bqrs.
Shutting down query evaluator.
Compiling query plan for /home/878352/Session-Management-Analyser/Django_Queries/Django-auth-is-used-check/django_library_used_check.ql.
[1/1] Found in cache: /home/878352/Session-Management-Analyser/Django_Queries/Django-auth-is-used-check/django_library_used_check.ql.
Starting evaluation of asterius27/codeql-extra-queries-python/Django_Queries/Django-auth-is-used-check/django_library_used_check.ql.
[1/1 eval 4.1s] Evaluation done; writing results to /home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/repositories/Django/thread4/raatmarien_my-qr.art/raatmarien-my-qr.art-26a4492-results/django_library_used_check.bqrs.
Shutting down query evaluator.
file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/python-single-analysis.js:97
        throw new Error("None of the supported libraries/frameworks is used");
              ^

Error: None of the supported libraries/frameworks is used
    at pythonAnalysis (file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/python-single-analysis.js:97:15)
    at file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/app.js:57:5
    at ModuleJob.run (node:internal/modules/esm/module_job:217:25)
    at async ModuleLoader.import (node:internal/modules/esm/loader:316:24)
    at async loadESM (node:internal/process/esm_loader:34:7)
    at async handleMainPromise (node:internal/modules/run_main:66:12)

Node.js v20.8.0

Time taken to run the queries on ./repositories/Django/thread4/raatmarien_my-qr.art/raatmarien-my-qr.art-26a4492: 114.264 seconds.
Time taken to run the queries on ./repositories/Django/thread4/rafalp_Misago/rafalp-Misago-a494b8d: 1583.556 seconds.
Time taken to run the queries on ./repositories/Django/thread4/rapidbenjamin_django-ledger/rapidbenjamin-django-ledger-c26ce78: 1041.931 seconds.
Analysis failed for: ./repositories/Django/thread4/realsuayip_django-sozluk/realsuayip-django-sozluk-801439d Reason: Error: Command failed: npm start -- -s=./repositories/Django/thread4/realsuayip_django-sozluk/realsuayip-django-sozluk-801439d -l=python -t=11 -ct=4
Compiling query plan for /home/878352/Session-Management-Analyser/Flask_Queries/Flask-login-is-used-check/flask_library_used_check.ql.
[1/1] Found in cache: /home/878352/Session-Management-Analyser/Flask_Queries/Flask-login-is-used-check/flask_library_used_check.ql.
Starting evaluation of asterius27/codeql-extra-queries-python/Flask_Queries/Flask-login-is-used-check/flask_library_used_check.ql.
[1/1 eval 33s] Evaluation done; writing results to /home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/repositories/Django/thread4/realsuayip_django-sozluk/realsuayip-django-sozluk-801439d-results/flask_library_used_check.bqrs.
Shutting down query evaluator.
Compiling query plan for /home/878352/Session-Management-Analyser/Django_Queries/Django-auth-is-used-check/django_library_used_check.ql.
[1/1] Found in cache: /home/878352/Session-Management-Analyser/Django_Queries/Django-auth-is-used-check/django_library_used_check.ql.
Starting evaluation of asterius27/codeql-extra-queries-python/Django_Queries/Django-auth-is-used-check/django_library_used_check.ql.
[1/1 eval 6.8s] Evaluation done; writing results to /home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/repositories/Django/thread4/realsuayip_django-sozluk/realsuayip-django-sozluk-801439d-results/django_library_used_check.bqrs.
Shutting down query evaluator.
Compiling query plan for /home/878352/Session-Management-Analyser/Django_Queries/Custom-session-engine/custom_session_engine.ql.
[1/1] Found in cache: /home/878352/Session-Management-Analyser/Django_Queries/Custom-session-engine/custom_session_engine.ql.
Starting evaluation of asterius27/codeql-extra-queries-python/Django_Queries/Custom-session-engine/custom_session_engine.ql.
[1/1 eval 13s] Evaluation done; writing results to /home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/repositories/Django/thread4/realsuayip_django-sozluk/realsuayip-django-sozluk-801439d-results/custom_session_engine.bqrs.
Shutting down query evaluator.
file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/python-single-analysis.js:144
            throw new Error("Using a custom session engine, so the analysis won't be run");
                  ^

Error: Using a custom session engine, so the analysis won't be run
    at pythonAnalysis (file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/python-single-analysis.js:144:19)
    at file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/app.js:57:5
    at ModuleJob.run (node:internal/modules/esm/module_job:217:25)
    at async ModuleLoader.import (node:internal/modules/esm/loader:316:24)
    at async loadESM (node:internal/process/esm_loader:34:7)
    at async handleMainPromise (node:internal/modules/run_main:66:12)

Node.js v20.8.0

Time taken to run the queries on ./repositories/Django/thread4/realsuayip_django-sozluk/realsuayip-django-sozluk-801439d: 132.444 seconds.
Analysis failed for: ./repositories/Django/thread4/recommend-games_recommend-games-server/recommend-games-recommend-games-server-639104c Reason: Error: Command failed: npm start -- -s=./repositories/Django/thread4/recommend-games_recommend-games-server/recommend-games-recommend-games-server-639104c -l=python -t=11 -ct=4
Compiling query plan for /home/878352/Session-Management-Analyser/Flask_Queries/Flask-login-is-used-check/flask_library_used_check.ql.
[1/1] Found in cache: /home/878352/Session-Management-Analyser/Flask_Queries/Flask-login-is-used-check/flask_library_used_check.ql.
Starting evaluation of asterius27/codeql-extra-queries-python/Flask_Queries/Flask-login-is-used-check/flask_library_used_check.ql.
[1/1 eval 55.1s] Evaluation done; writing results to /home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/repositories/Django/thread4/recommend-games_recommend-games-server/recommend-games-recommend-games-server-639104c-results/flask_library_used_check.bqrs.
Shutting down query evaluator.
Compiling query plan for /home/878352/Session-Management-Analyser/Django_Queries/Django-auth-is-used-check/django_library_used_check.ql.
[1/1] Found in cache: /home/878352/Session-Management-Analyser/Django_Queries/Django-auth-is-used-check/django_library_used_check.ql.
Starting evaluation of asterius27/codeql-extra-queries-python/Django_Queries/Django-auth-is-used-check/django_library_used_check.ql.
[1/1 eval 6.4s] Evaluation done; writing results to /home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/repositories/Django/thread4/recommend-games_recommend-games-server/recommend-games-recommend-games-server-639104c-results/django_library_used_check.bqrs.
Shutting down query evaluator.
file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/python-single-analysis.js:97
        throw new Error("None of the supported libraries/frameworks is used");
              ^

Error: None of the supported libraries/frameworks is used
    at pythonAnalysis (file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/python-single-analysis.js:97:15)
    at file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/app.js:57:5
    at ModuleJob.run (node:internal/modules/esm/module_job:217:25)
    at async ModuleLoader.import (node:internal/modules/esm/loader:316:24)
    at async loadESM (node:internal/process/esm_loader:34:7)
    at async handleMainPromise (node:internal/modules/run_main:66:12)

Node.js v20.8.0

Time taken to run the queries on ./repositories/Django/thread4/recommend-games_recommend-games-server/recommend-games-recommend-games-server-639104c: 142.147 seconds.
Analysis failed for: ./repositories/Django/thread4/route4me_route4me-curl/route4me-route4me-curl-985748e Reason: Error: Command failed: codeql database create ./repositories/Django/thread4/route4me_route4me-curl/route4me-route4me-curl-985748e-database --language=python --source-root ./repositories/Django/thread4/route4me_route4me-curl/route4me-route4me-curl-985748e --threads=11
Initializing database at /home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/repositories/Django/thread4/route4me_route4me-curl/route4me-route4me-curl-985748e-database.
Running build command: []
Finalizing database at /home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/repositories/Django/thread4/route4me_route4me-curl/route4me-route4me-curl-985748e-database.
CodeQL did not detect any code written in languages supported by CodeQL. This can occur if the specified build commands failed to compile or process any code.
 - Confirm that there is some source code for the specified language in the project.
 - For codebases written in Go, JavaScript, TypeScript, and Python, do not specify 
   an explicit --command.
 - For other languages, the --command must specify a "clean" build which compiles 
   all the source code files without reusing existing build artefacts.

Time taken to run the queries on ./repositories/Django/thread4/route4me_route4me-curl/route4me-route4me-curl-985748e: 5.276 seconds.
Time taken to run the queries on ./repositories/Django/thread4/saatvikj_ConferenceApp/saatvikj-ConferenceApp-69cc2f2: 839.342 seconds.
Time taken to run the queries on ./repositories/Django/thread4/sflems_led-board-manager/sflems-led-board-manager-12a1b22: 905.062 seconds.
Analysis failed for: ./repositories/Django/thread4/shayan-cyber_DotSlashProj/shayan-cyber-DotSlashProj-41b58b4 Reason: Error: Command failed: npm start -- -s=./repositories/Django/thread4/shayan-cyber_DotSlashProj/shayan-cyber-DotSlashProj-41b58b4 -l=python -t=11 -ct=4
Compiling query plan for /home/878352/Session-Management-Analyser/Flask_Queries/Flask-login-is-used-check/flask_library_used_check.ql.
[1/1] Found in cache: /home/878352/Session-Management-Analyser/Flask_Queries/Flask-login-is-used-check/flask_library_used_check.ql.
Starting evaluation of asterius27/codeql-extra-queries-python/Flask_Queries/Flask-login-is-used-check/flask_library_used_check.ql.
[1/1 eval 29.6s] Evaluation done; writing results to /home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/repositories/Django/thread4/shayan-cyber_DotSlashProj/shayan-cyber-DotSlashProj-41b58b4-results/flask_library_used_check.bqrs.
Shutting down query evaluator.
Compiling query plan for /home/878352/Session-Management-Analyser/Django_Queries/Django-auth-is-used-check/django_library_used_check.ql.
[1/1] Found in cache: /home/878352/Session-Management-Analyser/Django_Queries/Django-auth-is-used-check/django_library_used_check.ql.
Starting evaluation of asterius27/codeql-extra-queries-python/Django_Queries/Django-auth-is-used-check/django_library_used_check.ql.
[1/1 eval 6.2s] Evaluation done; writing results to /home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/repositories/Django/thread4/shayan-cyber_DotSlashProj/shayan-cyber-DotSlashProj-41b58b4-results/django_library_used_check.bqrs.
Shutting down query evaluator.
file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/python-single-analysis.js:97
        throw new Error("None of the supported libraries/frameworks is used");
              ^

Error: None of the supported libraries/frameworks is used
    at pythonAnalysis (file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/python-single-analysis.js:97:15)
    at file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/app.js:57:5
    at ModuleJob.run (node:internal/modules/esm/module_job:217:25)
    at async ModuleLoader.import (node:internal/modules/esm/loader:316:24)
    at async loadESM (node:internal/process/esm_loader:34:7)
    at async handleMainPromise (node:internal/modules/run_main:66:12)

Node.js v20.8.0

Time taken to run the queries on ./repositories/Django/thread4/shayan-cyber_DotSlashProj/shayan-cyber-DotSlashProj-41b58b4: 87.172 seconds.
Analysis failed for: ./repositories/Django/thread4/shhubhxm_HousePricePrediction_website/shhubhxm-HousePricePrediction_website-8d45a3f Reason: Error: Command failed: npm start -- -s=./repositories/Django/thread4/shhubhxm_HousePricePrediction_website/shhubhxm-HousePricePrediction_website-8d45a3f -l=python -t=11 -ct=4
Compiling query plan for /home/878352/Session-Management-Analyser/Flask_Queries/Flask-login-is-used-check/flask_library_used_check.ql.
[1/1] Found in cache: /home/878352/Session-Management-Analyser/Flask_Queries/Flask-login-is-used-check/flask_library_used_check.ql.
Starting evaluation of asterius27/codeql-extra-queries-python/Flask_Queries/Flask-login-is-used-check/flask_library_used_check.ql.
[1/1 eval 20.8s] Evaluation done; writing results to /home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/repositories/Django/thread4/shhubhxm_HousePricePrediction_website/shhubhxm-HousePricePrediction_website-8d45a3f-results/flask_library_used_check.bqrs.
Shutting down query evaluator.
Compiling query plan for /home/878352/Session-Management-Analyser/Django_Queries/Django-auth-is-used-check/django_library_used_check.ql.
[1/1] Found in cache: /home/878352/Session-Management-Analyser/Django_Queries/Django-auth-is-used-check/django_library_used_check.ql.
Starting evaluation of asterius27/codeql-extra-queries-python/Django_Queries/Django-auth-is-used-check/django_library_used_check.ql.
[1/1 eval 3.3s] Evaluation done; writing results to /home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/repositories/Django/thread4/shhubhxm_HousePricePrediction_website/shhubhxm-HousePricePrediction_website-8d45a3f-results/django_library_used_check.bqrs.
Shutting down query evaluator.
file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/python-single-analysis.js:97
        throw new Error("None of the supported libraries/frameworks is used");
              ^

Error: None of the supported libraries/frameworks is used
    at pythonAnalysis (file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/python-single-analysis.js:97:15)
    at file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/app.js:57:5
    at ModuleJob.run (node:internal/modules/esm/module_job:217:25)
    at async ModuleLoader.import (node:internal/modules/esm/loader:316:24)
    at async loadESM (node:internal/process/esm_loader:34:7)
    at async handleMainPromise (node:internal/modules/run_main:66:12)

Node.js v20.8.0

Time taken to run the queries on ./repositories/Django/thread4/shhubhxm_HousePricePrediction_website/shhubhxm-HousePricePrediction_website-8d45a3f: 68.689 seconds.
Analysis failed for: ./repositories/Django/thread4/sorbayhq_sorbay/sorbayhq-sorbay-0e349f3 Reason: Error: Command failed: npm start -- -s=./repositories/Django/thread4/sorbayhq_sorbay/sorbayhq-sorbay-0e349f3 -l=python -t=11 -ct=4
Compiling query plan for /home/878352/Session-Management-Analyser/Flask_Queries/Flask-login-is-used-check/flask_library_used_check.ql.
[1/1] Found in cache: /home/878352/Session-Management-Analyser/Flask_Queries/Flask-login-is-used-check/flask_library_used_check.ql.
Starting evaluation of asterius27/codeql-extra-queries-python/Flask_Queries/Flask-login-is-used-check/flask_library_used_check.ql.
[1/1 eval 22.4s] Evaluation done; writing results to /home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/repositories/Django/thread4/sorbayhq_sorbay/sorbayhq-sorbay-0e349f3-results/flask_library_used_check.bqrs.
Shutting down query evaluator.
Compiling query plan for /home/878352/Session-Management-Analyser/Django_Queries/Django-auth-is-used-check/django_library_used_check.ql.
[1/1] Found in cache: /home/878352/Session-Management-Analyser/Django_Queries/Django-auth-is-used-check/django_library_used_check.ql.
Starting evaluation of asterius27/codeql-extra-queries-python/Django_Queries/Django-auth-is-used-check/django_library_used_check.ql.
[1/1 eval 3.2s] Evaluation done; writing results to /home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/repositories/Django/thread4/sorbayhq_sorbay/sorbayhq-sorbay-0e349f3-results/django_library_used_check.bqrs.
Shutting down query evaluator.
file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/python-single-analysis.js:97
        throw new Error("None of the supported libraries/frameworks is used");
              ^

Error: None of the supported libraries/frameworks is used
    at pythonAnalysis (file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/python-single-analysis.js:97:15)
    at file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/app.js:57:5
    at ModuleJob.run (node:internal/modules/esm/module_job:217:25)
    at async ModuleLoader.import (node:internal/modules/esm/loader:316:24)
    at async loadESM (node:internal/process/esm_loader:34:7)
    at async handleMainPromise (node:internal/modules/run_main:66:12)

Node.js v20.8.0

Time taken to run the queries on ./repositories/Django/thread4/sorbayhq_sorbay/sorbayhq-sorbay-0e349f3: 72.214 seconds.
Analysis failed for: ./repositories/Django/thread4/soynatan_django-easy-audit/soynatan-django-easy-audit-9e04815 Reason: Error: Command failed: npm start -- -s=./repositories/Django/thread4/soynatan_django-easy-audit/soynatan-django-easy-audit-9e04815 -l=python -t=11 -ct=4
Compiling query plan for /home/878352/Session-Management-Analyser/Flask_Queries/Flask-login-is-used-check/flask_library_used_check.ql.
[1/1] Found in cache: /home/878352/Session-Management-Analyser/Flask_Queries/Flask-login-is-used-check/flask_library_used_check.ql.
Starting evaluation of asterius27/codeql-extra-queries-python/Flask_Queries/Flask-login-is-used-check/flask_library_used_check.ql.
[1/1 eval 20.1s] Evaluation done; writing results to /home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/repositories/Django/thread4/soynatan_django-easy-audit/soynatan-django-easy-audit-9e04815-results/flask_library_used_check.bqrs.
Shutting down query evaluator.
Compiling query plan for /home/878352/Session-Management-Analyser/Django_Queries/Django-auth-is-used-check/django_library_used_check.ql.
[1/1] Found in cache: /home/878352/Session-Management-Analyser/Django_Queries/Django-auth-is-used-check/django_library_used_check.ql.
Starting evaluation of asterius27/codeql-extra-queries-python/Django_Queries/Django-auth-is-used-check/django_library_used_check.ql.
[1/1 eval 3.7s] Evaluation done; writing results to /home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/repositories/Django/thread4/soynatan_django-easy-audit/soynatan-django-easy-audit-9e04815-results/django_library_used_check.bqrs.
Shutting down query evaluator.
file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/python-single-analysis.js:97
        throw new Error("None of the supported libraries/frameworks is used");
              ^

Error: None of the supported libraries/frameworks is used
    at pythonAnalysis (file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/python-single-analysis.js:97:15)
    at file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/app.js:57:5
    at ModuleJob.run (node:internal/modules/esm/module_job:217:25)
    at async ModuleLoader.import (node:internal/modules/esm/loader:316:24)
    at async loadESM (node:internal/process/esm_loader:34:7)
    at async handleMainPromise (node:internal/modules/run_main:66:12)

Node.js v20.8.0

Time taken to run the queries on ./repositories/Django/thread4/soynatan_django-easy-audit/soynatan-django-easy-audit-9e04815: 70.819 seconds.
Analysis failed for: ./repositories/Django/thread4/stackhpc_azimuth/stackhpc-azimuth-46d3323 Reason: Error: Command failed: npm start -- -s=./repositories/Django/thread4/stackhpc_azimuth/stackhpc-azimuth-46d3323 -l=python -t=11 -ct=4
Compiling query plan for /home/878352/Session-Management-Analyser/Flask_Queries/Flask-login-is-used-check/flask_library_used_check.ql.
[1/1] Found in cache: /home/878352/Session-Management-Analyser/Flask_Queries/Flask-login-is-used-check/flask_library_used_check.ql.
Starting evaluation of asterius27/codeql-extra-queries-python/Flask_Queries/Flask-login-is-used-check/flask_library_used_check.ql.
[1/1 eval 31.6s] Evaluation done; writing results to /home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/repositories/Django/thread4/stackhpc_azimuth/stackhpc-azimuth-46d3323-results/flask_library_used_check.bqrs.
Shutting down query evaluator.
Compiling query plan for /home/878352/Session-Management-Analyser/Django_Queries/Django-auth-is-used-check/django_library_used_check.ql.
[1/1] Found in cache: /home/878352/Session-Management-Analyser/Django_Queries/Django-auth-is-used-check/django_library_used_check.ql.
Starting evaluation of asterius27/codeql-extra-queries-python/Django_Queries/Django-auth-is-used-check/django_library_used_check.ql.
[1/1 eval 4.6s] Evaluation done; writing results to /home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/repositories/Django/thread4/stackhpc_azimuth/stackhpc-azimuth-46d3323-results/django_library_used_check.bqrs.
Shutting down query evaluator.
file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/python-single-analysis.js:97
        throw new Error("None of the supported libraries/frameworks is used");
              ^

Error: None of the supported libraries/frameworks is used
    at pythonAnalysis (file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/python-single-analysis.js:97:15)
    at file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/app.js:57:5
    at ModuleJob.run (node:internal/modules/esm/module_job:217:25)
    at async ModuleLoader.import (node:internal/modules/esm/loader:316:24)
    at async loadESM (node:internal/process/esm_loader:34:7)
    at async handleMainPromise (node:internal/modules/run_main:66:12)

Node.js v20.8.0

Time taken to run the queries on ./repositories/Django/thread4/stackhpc_azimuth/stackhpc-azimuth-46d3323: 93.441 seconds.
Time taken to run the queries on ./repositories/Django/thread4/stephanekirsch_e-colle/stephanekirsch-e-colle-7d07f2b: 674.094 seconds.
Analysis failed for: ./repositories/Django/thread4/stuartelimu_vaccination-centres/stuartelimu-vaccination-centres-6df14f1 Reason: Error: Command failed: npm start -- -s=./repositories/Django/thread4/stuartelimu_vaccination-centres/stuartelimu-vaccination-centres-6df14f1 -l=python -t=11 -ct=4
Compiling query plan for /home/878352/Session-Management-Analyser/Flask_Queries/Flask-login-is-used-check/flask_library_used_check.ql.
[1/1] Found in cache: /home/878352/Session-Management-Analyser/Flask_Queries/Flask-login-is-used-check/flask_library_used_check.ql.
Starting evaluation of asterius27/codeql-extra-queries-python/Flask_Queries/Flask-login-is-used-check/flask_library_used_check.ql.
[1/1 eval 18.2s] Evaluation done; writing results to /home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/repositories/Django/thread4/stuartelimu_vaccination-centres/stuartelimu-vaccination-centres-6df14f1-results/flask_library_used_check.bqrs.
Shutting down query evaluator.
Compiling query plan for /home/878352/Session-Management-Analyser/Django_Queries/Django-auth-is-used-check/django_library_used_check.ql.
[1/1] Found in cache: /home/878352/Session-Management-Analyser/Django_Queries/Django-auth-is-used-check/django_library_used_check.ql.
Starting evaluation of asterius27/codeql-extra-queries-python/Django_Queries/Django-auth-is-used-check/django_library_used_check.ql.
[1/1 eval 3.7s] Evaluation done; writing results to /home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/repositories/Django/thread4/stuartelimu_vaccination-centres/stuartelimu-vaccination-centres-6df14f1-results/django_library_used_check.bqrs.
Shutting down query evaluator.
file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/python-single-analysis.js:97
        throw new Error("None of the supported libraries/frameworks is used");
              ^

Error: None of the supported libraries/frameworks is used
    at pythonAnalysis (file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/python-single-analysis.js:97:15)
    at file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/app.js:57:5
    at ModuleJob.run (node:internal/modules/esm/module_job:217:25)
    at async ModuleLoader.import (node:internal/modules/esm/loader:316:24)
    at async loadESM (node:internal/process/esm_loader:34:7)
    at async handleMainPromise (node:internal/modules/run_main:66:12)

Node.js v20.8.0

Time taken to run the queries on ./repositories/Django/thread4/stuartelimu_vaccination-centres/stuartelimu-vaccination-centres-6df14f1: 54.842 seconds.
Time taken to run the queries on ./repositories/Django/thread4/sy159_gmx-cms/sy159-gmx-cms-34adf48: 520.013 seconds.
Time taken to run the queries on ./repositories/Django/thread4/themoonsheep_moonsheep/themoonsheep-moonsheep-d03b965: 486.118 seconds.
Analysis failed for: ./repositories/Django/thread4/tira-io_tira/tira-io-tira-ff0eabe Reason: Error: Command failed: npm start -- -s=./repositories/Django/thread4/tira-io_tira/tira-io-tira-ff0eabe -l=python -t=11 -ct=4
Compiling query plan for /home/878352/Session-Management-Analyser/Flask_Queries/Flask-login-is-used-check/flask_library_used_check.ql.
[1/1] Found in cache: /home/878352/Session-Management-Analyser/Flask_Queries/Flask-login-is-used-check/flask_library_used_check.ql.
Starting evaluation of asterius27/codeql-extra-queries-python/Flask_Queries/Flask-login-is-used-check/flask_library_used_check.ql.
[1/1 eval 28.9s] Evaluation done; writing results to /home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/repositories/Django/thread4/tira-io_tira/tira-io-tira-ff0eabe-results/flask_library_used_check.bqrs.
Shutting down query evaluator.
Compiling query plan for /home/878352/Session-Management-Analyser/Django_Queries/Django-auth-is-used-check/django_library_used_check.ql.
[1/1] Found in cache: /home/878352/Session-Management-Analyser/Django_Queries/Django-auth-is-used-check/django_library_used_check.ql.
Starting evaluation of asterius27/codeql-extra-queries-python/Django_Queries/Django-auth-is-used-check/django_library_used_check.ql.
[1/1 eval 3.8s] Evaluation done; writing results to /home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/repositories/Django/thread4/tira-io_tira/tira-io-tira-ff0eabe-results/django_library_used_check.bqrs.
Shutting down query evaluator.
file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/python-single-analysis.js:97
        throw new Error("None of the supported libraries/frameworks is used");
              ^

Error: None of the supported libraries/frameworks is used
    at pythonAnalysis (file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/python-single-analysis.js:97:15)
    at file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/app.js:57:5
    at ModuleJob.run (node:internal/modules/esm/module_job:217:25)
    at async ModuleLoader.import (node:internal/modules/esm/loader:316:24)
    at async loadESM (node:internal/process/esm_loader:34:7)
    at async handleMainPromise (node:internal/modules/run_main:66:12)

Node.js v20.8.0

Time taken to run the queries on ./repositories/Django/thread4/tira-io_tira/tira-io-tira-ff0eabe: 76.419 seconds.
Analysis failed for: ./repositories/Django/thread4/torchbox_rca-wagtail-2019/torchbox-rca-wagtail-2019-b820412 Reason: Error: Command failed: npm start -- -s=./repositories/Django/thread4/torchbox_rca-wagtail-2019/torchbox-rca-wagtail-2019-b820412 -l=python -t=11 -ct=4
Compiling query plan for /home/878352/Session-Management-Analyser/Flask_Queries/Flask-login-is-used-check/flask_library_used_check.ql.
[1/1] Found in cache: /home/878352/Session-Management-Analyser/Flask_Queries/Flask-login-is-used-check/flask_library_used_check.ql.
Starting evaluation of asterius27/codeql-extra-queries-python/Flask_Queries/Flask-login-is-used-check/flask_library_used_check.ql.
[1/1 eval 17.3s] Evaluation done; writing results to /home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/repositories/Django/thread4/torchbox_rca-wagtail-2019/torchbox-rca-wagtail-2019-b820412-results/flask_library_used_check.bqrs.
Shutting down query evaluator.
Compiling query plan for /home/878352/Session-Management-Analyser/Django_Queries/Django-auth-is-used-check/django_library_used_check.ql.
[1/1] Found in cache: /home/878352/Session-Management-Analyser/Django_Queries/Django-auth-is-used-check/django_library_used_check.ql.
Starting evaluation of asterius27/codeql-extra-queries-python/Django_Queries/Django-auth-is-used-check/django_library_used_check.ql.
[1/1 eval 3.5s] Evaluation done; writing results to /home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/repositories/Django/thread4/torchbox_rca-wagtail-2019/torchbox-rca-wagtail-2019-b820412-results/django_library_used_check.bqrs.
Shutting down query evaluator.
file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/python-single-analysis.js:97
        throw new Error("None of the supported libraries/frameworks is used");
              ^

Error: None of the supported libraries/frameworks is used
    at pythonAnalysis (file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/python-single-analysis.js:97:15)
    at file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/app.js:57:5
    at ModuleJob.run (node:internal/modules/esm/module_job:217:25)
    at async ModuleLoader.import (node:internal/modules/esm/loader:316:24)
    at async loadESM (node:internal/process/esm_loader:34:7)
    at async handleMainPromise (node:internal/modules/run_main:66:12)

Node.js v20.8.0

Time taken to run the queries on ./repositories/Django/thread4/torchbox_rca-wagtail-2019/torchbox-rca-wagtail-2019-b820412: 54.218 seconds.
Analysis failed for: ./repositories/Django/thread4/uwblueprint_project-read-backend/uwblueprint-project-read-backend-bf3093f Reason: Error: Command failed: npm start -- -s=./repositories/Django/thread4/uwblueprint_project-read-backend/uwblueprint-project-read-backend-bf3093f -l=python -t=11 -ct=4
Compiling query plan for /home/878352/Session-Management-Analyser/Flask_Queries/Flask-login-is-used-check/flask_library_used_check.ql.
[1/1] Found in cache: /home/878352/Session-Management-Analyser/Flask_Queries/Flask-login-is-used-check/flask_library_used_check.ql.
Starting evaluation of asterius27/codeql-extra-queries-python/Flask_Queries/Flask-login-is-used-check/flask_library_used_check.ql.
[1/1 eval 17.6s] Evaluation done; writing results to /home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/repositories/Django/thread4/uwblueprint_project-read-backend/uwblueprint-project-read-backend-bf3093f-results/flask_library_used_check.bqrs.
Shutting down query evaluator.
Compiling query plan for /home/878352/Session-Management-Analyser/Django_Queries/Django-auth-is-used-check/django_library_used_check.ql.
[1/1] Found in cache: /home/878352/Session-Management-Analyser/Django_Queries/Django-auth-is-used-check/django_library_used_check.ql.
Starting evaluation of asterius27/codeql-extra-queries-python/Django_Queries/Django-auth-is-used-check/django_library_used_check.ql.
[1/1 eval 3.1s] Evaluation done; writing results to /home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/repositories/Django/thread4/uwblueprint_project-read-backend/uwblueprint-project-read-backend-bf3093f-results/django_library_used_check.bqrs.
Shutting down query evaluator.
file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/python-single-analysis.js:97
        throw new Error("None of the supported libraries/frameworks is used");
              ^

Error: None of the supported libraries/frameworks is used
    at pythonAnalysis (file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/python-single-analysis.js:97:15)
    at file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/app.js:57:5
    at ModuleJob.run (node:internal/modules/esm/module_job:217:25)
    at async ModuleLoader.import (node:internal/modules/esm/loader:316:24)
    at async loadESM (node:internal/process/esm_loader:34:7)
    at async handleMainPromise (node:internal/modules/run_main:66:12)

Node.js v20.8.0

Time taken to run the queries on ./repositories/Django/thread4/uwblueprint_project-read-backend/uwblueprint-project-read-backend-bf3093f: 51.269 seconds.
Analysis failed for: ./repositories/Django/thread4/varunmamtora06_FernGallery/varunmamtora06-FernGallery-3e90a2c Reason: Error: Command failed: npm start -- -s=./repositories/Django/thread4/varunmamtora06_FernGallery/varunmamtora06-FernGallery-3e90a2c -l=python -t=11 -ct=4
Compiling query plan for /home/878352/Session-Management-Analyser/Flask_Queries/Flask-login-is-used-check/flask_library_used_check.ql.
[1/1] Found in cache: /home/878352/Session-Management-Analyser/Flask_Queries/Flask-login-is-used-check/flask_library_used_check.ql.
Starting evaluation of asterius27/codeql-extra-queries-python/Flask_Queries/Flask-login-is-used-check/flask_library_used_check.ql.
[1/1 eval 22s] Evaluation done; writing results to /home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/repositories/Django/thread4/varunmamtora06_FernGallery/varunmamtora06-FernGallery-3e90a2c-results/flask_library_used_check.bqrs.
Shutting down query evaluator.
Compiling query plan for /home/878352/Session-Management-Analyser/Django_Queries/Django-auth-is-used-check/django_library_used_check.ql.
[1/1] Found in cache: /home/878352/Session-Management-Analyser/Django_Queries/Django-auth-is-used-check/django_library_used_check.ql.
Starting evaluation of asterius27/codeql-extra-queries-python/Django_Queries/Django-auth-is-used-check/django_library_used_check.ql.
[1/1 eval 4s] Evaluation done; writing results to /home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/repositories/Django/thread4/varunmamtora06_FernGallery/varunmamtora06-FernGallery-3e90a2c-results/django_library_used_check.bqrs.
Shutting down query evaluator.
file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/python-single-analysis.js:97
        throw new Error("None of the supported libraries/frameworks is used");
              ^

Error: None of the supported libraries/frameworks is used
    at pythonAnalysis (file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/python-single-analysis.js:97:15)
    at file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/app.js:57:5
    at ModuleJob.run (node:internal/modules/esm/module_job:217:25)
    at async ModuleLoader.import (node:internal/modules/esm/loader:316:24)
    at async loadESM (node:internal/process/esm_loader:34:7)
    at async handleMainPromise (node:internal/modules/run_main:66:12)

Node.js v20.8.0

Time taken to run the queries on ./repositories/Django/thread4/varunmamtora06_FernGallery/varunmamtora06-FernGallery-3e90a2c: 61.019 seconds.
Time taken to run the queries on ./repositories/Django/thread4/veryacademy_django-ecommerce-project/veryacademy-django-ecommerce-project-85d194e: 337.343 seconds.
Time taken to run the queries on ./repositories/Django/thread4/wallysalami_gamified-education/wallysalami-gamified-education-2774d0e: 319.251 seconds.
Analysis failed for: ./repositories/Django/thread4/webmalc_nativecards-server/webmalc-nativecards-server-3baff11 Reason: Error: Command failed: npm start -- -s=./repositories/Django/thread4/webmalc_nativecards-server/webmalc-nativecards-server-3baff11 -l=python -t=11 -ct=4
Compiling query plan for /home/878352/Session-Management-Analyser/Flask_Queries/Flask-login-is-used-check/flask_library_used_check.ql.
[1/1] Found in cache: /home/878352/Session-Management-Analyser/Flask_Queries/Flask-login-is-used-check/flask_library_used_check.ql.
Starting evaluation of asterius27/codeql-extra-queries-python/Flask_Queries/Flask-login-is-used-check/flask_library_used_check.ql.
[1/1 eval 15.4s] Evaluation done; writing results to /home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/repositories/Django/thread4/webmalc_nativecards-server/webmalc-nativecards-server-3baff11-results/flask_library_used_check.bqrs.
Shutting down query evaluator.
Compiling query plan for /home/878352/Session-Management-Analyser/Django_Queries/Django-auth-is-used-check/django_library_used_check.ql.
[1/1] Found in cache: /home/878352/Session-Management-Analyser/Django_Queries/Django-auth-is-used-check/django_library_used_check.ql.
Starting evaluation of asterius27/codeql-extra-queries-python/Django_Queries/Django-auth-is-used-check/django_library_used_check.ql.
[1/1 eval 2.9s] Evaluation done; writing results to /home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/repositories/Django/thread4/webmalc_nativecards-server/webmalc-nativecards-server-3baff11-results/django_library_used_check.bqrs.
Shutting down query evaluator.
file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/python-single-analysis.js:97
        throw new Error("None of the supported libraries/frameworks is used");
              ^

Error: None of the supported libraries/frameworks is used
    at pythonAnalysis (file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/python-single-analysis.js:97:15)
    at file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/app.js:57:5
    at ModuleJob.run (node:internal/modules/esm/module_job:217:25)
    at async ModuleLoader.import (node:internal/modules/esm/loader:316:24)
    at async loadESM (node:internal/process/esm_loader:34:7)
    at async handleMainPromise (node:internal/modules/run_main:66:12)

Node.js v20.8.0

Time taken to run the queries on ./repositories/Django/thread4/webmalc_nativecards-server/webmalc-nativecards-server-3baff11: 45.389 seconds.
Time taken to run the queries on ./repositories/Django/thread4/wevote_WeVoteServer/wevote-WeVoteServer-e14e6b9: 677.182 seconds.
Time taken to run the queries on ./repositories/Django/thread4/womenandcolor_women-and-color-backend/womenandcolor-women-and-color-backend-60c3a5d: 364.015 seconds.
Time taken to run the queries on ./repositories/Django/thread4/xhongc_dj-chat/xhongc-dj-chat-2e88425: 364.51 seconds.
Time taken to run the queries on ./repositories/Django/thread4/yverdon_geocity/yverdon-geocity-94a68ee: 431.142 seconds.
Analysis failed for: ./repositories/Django/thread4/zetonteam_zeton_django/zetonteam-zeton_django-c43364e Reason: Error: Command failed: npm start -- -s=./repositories/Django/thread4/zetonteam_zeton_django/zetonteam-zeton_django-c43364e -l=python -t=11 -ct=4
Compiling query plan for /home/878352/Session-Management-Analyser/Flask_Queries/Flask-login-is-used-check/flask_library_used_check.ql.
[1/1] Found in cache: /home/878352/Session-Management-Analyser/Flask_Queries/Flask-login-is-used-check/flask_library_used_check.ql.
Starting evaluation of asterius27/codeql-extra-queries-python/Flask_Queries/Flask-login-is-used-check/flask_library_used_check.ql.
[1/1 eval 12.7s] Evaluation done; writing results to /home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/repositories/Django/thread4/zetonteam_zeton_django/zetonteam-zeton_django-c43364e-results/flask_library_used_check.bqrs.
Shutting down query evaluator.
Compiling query plan for /home/878352/Session-Management-Analyser/Django_Queries/Django-auth-is-used-check/django_library_used_check.ql.
[1/1] Found in cache: /home/878352/Session-Management-Analyser/Django_Queries/Django-auth-is-used-check/django_library_used_check.ql.
Starting evaluation of asterius27/codeql-extra-queries-python/Django_Queries/Django-auth-is-used-check/django_library_used_check.ql.
[1/1 eval 2.2s] Evaluation done; writing results to /home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/repositories/Django/thread4/zetonteam_zeton_django/zetonteam-zeton_django-c43364e-results/django_library_used_check.bqrs.
Shutting down query evaluator.
file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/python-single-analysis.js:97
        throw new Error("None of the supported libraries/frameworks is used");
              ^

Error: None of the supported libraries/frameworks is used
    at pythonAnalysis (file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/python-single-analysis.js:97:15)
    at file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/app.js:57:5
    at ModuleJob.run (node:internal/modules/esm/module_job:217:25)
    at async ModuleLoader.import (node:internal/modules/esm/loader:316:24)
    at async loadESM (node:internal/process/esm_loader:34:7)
    at async handleMainPromise (node:internal/modules/run_main:66:12)

Node.js v20.8.0

Time taken to run the queries on ./repositories/Django/thread4/zetonteam_zeton_django/zetonteam-zeton_django-c43364e: 35.722 seconds.
Analysis failed for: ./repositories/Django/thread4/zhangjian-ai_chatroom/zhangjian-ai-chatroom-d667850 Reason: Error: Command failed: npm start -- -s=./repositories/Django/thread4/zhangjian-ai_chatroom/zhangjian-ai-chatroom-d667850 -l=python -t=11 -ct=4
Compiling query plan for /home/878352/Session-Management-Analyser/Flask_Queries/Flask-login-is-used-check/flask_library_used_check.ql.
[1/1] Found in cache: /home/878352/Session-Management-Analyser/Flask_Queries/Flask-login-is-used-check/flask_library_used_check.ql.
Starting evaluation of asterius27/codeql-extra-queries-python/Flask_Queries/Flask-login-is-used-check/flask_library_used_check.ql.
[1/1 eval 13s] Evaluation done; writing results to /home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/repositories/Django/thread4/zhangjian-ai_chatroom/zhangjian-ai-chatroom-d667850-results/flask_library_used_check.bqrs.
Shutting down query evaluator.
Compiling query plan for /home/878352/Session-Management-Analyser/Django_Queries/Django-auth-is-used-check/django_library_used_check.ql.
[1/1] Found in cache: /home/878352/Session-Management-Analyser/Django_Queries/Django-auth-is-used-check/django_library_used_check.ql.
Starting evaluation of asterius27/codeql-extra-queries-python/Django_Queries/Django-auth-is-used-check/django_library_used_check.ql.
[1/1 eval 2.2s] Evaluation done; writing results to /home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/repositories/Django/thread4/zhangjian-ai_chatroom/zhangjian-ai-chatroom-d667850-results/django_library_used_check.bqrs.
Shutting down query evaluator.
file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/python-single-analysis.js:97
        throw new Error("None of the supported libraries/frameworks is used");
              ^

Error: None of the supported libraries/frameworks is used
    at pythonAnalysis (file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/python-single-analysis.js:97:15)
    at file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/app.js:57:5
    at ModuleJob.run (node:internal/modules/esm/module_job:217:25)
    at async ModuleLoader.import (node:internal/modules/esm/loader:316:24)
    at async loadESM (node:internal/process/esm_loader:34:7)
    at async handleMainPromise (node:internal/modules/run_main:66:12)

Node.js v20.8.0

Time taken to run the queries on ./repositories/Django/thread4/zhangjian-ai_chatroom/zhangjian-ai-chatroom-d667850: 35.771 seconds.
Analysis failed for: ./repositories/Django/thread4/zuri-training_clovar_cms/zuri-training-clovar_cms-2eb2656 Reason: Error: Command failed: npm start -- -s=./repositories/Django/thread4/zuri-training_clovar_cms/zuri-training-clovar_cms-2eb2656 -l=python -t=11 -ct=4
Compiling query plan for /home/878352/Session-Management-Analyser/Flask_Queries/Flask-login-is-used-check/flask_library_used_check.ql.
[1/1] Found in cache: /home/878352/Session-Management-Analyser/Flask_Queries/Flask-login-is-used-check/flask_library_used_check.ql.
Starting evaluation of asterius27/codeql-extra-queries-python/Flask_Queries/Flask-login-is-used-check/flask_library_used_check.ql.
[1/1 eval 12.4s] Evaluation done; writing results to /home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/repositories/Django/thread4/zuri-training_clovar_cms/zuri-training-clovar_cms-2eb2656-results/flask_library_used_check.bqrs.
Shutting down query evaluator.
Compiling query plan for /home/878352/Session-Management-Analyser/Django_Queries/Django-auth-is-used-check/django_library_used_check.ql.
[1/1] Found in cache: /home/878352/Session-Management-Analyser/Django_Queries/Django-auth-is-used-check/django_library_used_check.ql.
Starting evaluation of asterius27/codeql-extra-queries-python/Django_Queries/Django-auth-is-used-check/django_library_used_check.ql.
[1/1 eval 2.3s] Evaluation done; writing results to /home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/repositories/Django/thread4/zuri-training_clovar_cms/zuri-training-clovar_cms-2eb2656-results/django_library_used_check.bqrs.
Shutting down query evaluator.
file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/python-single-analysis.js:97
        throw new Error("None of the supported libraries/frameworks is used");
              ^

Error: None of the supported libraries/frameworks is used
    at pythonAnalysis (file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/python-single-analysis.js:97:15)
    at file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/app.js:57:5
    at ModuleJob.run (node:internal/modules/esm/module_job:217:25)
    at async ModuleLoader.import (node:internal/modules/esm/loader:316:24)
    at async loadESM (node:internal/process/esm_loader:34:7)
    at async handleMainPromise (node:internal/modules/run_main:66:12)

Node.js v20.8.0

Time taken to run the queries on ./repositories/Django/thread4/zuri-training_clovar_cms/zuri-training-clovar_cms-2eb2656: 35.068 seconds.
Time taken to run the queries and generate the statistics: 171116.584 seconds.
